<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Glycine &#8211; LIS: Legume Information System</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="A collaborative, community resource to facilitate crop improvement by integrating genetic, genomic, and trait data across legume species.">
  <meta name="robots" content="all">
  <meta name="author" content="">
  
  
  <link rel="canonical" href="/collections/glycine/">

  
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-49655154-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-49655154-1');
</script>



  <!-- Custom CSS -->
  <link rel="stylesheet" href="/assets/uikit/dist/css/uikit.min.css" type="text/css" />
  <link rel="stylesheet" href="/assets/css/uikit-theme.css" type="text/css" />

  <!-- place globally-included stylesheet content here -->
<style>
  .species-image {
      float: right;
      margin: 10px;
  }
  .species-image img {
      border: 1px solid darkgreen;
  }
  .species-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .blog-image {
      float: right;
      margin: 10px;
  }
  .blog-image img {
      border: 1px solid gray;
  }
  .blog-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .temp {
      color: darkred;
      font-family: serif;
      font-weight: bold;
  }
</style>



  <!-- Custom JS -->
  <script src="/assets/uikit/dist/js/uikit.min.js"></script>
  <script src="/assets/uikit/dist/js/uikit-icons.min.js"></script>

  <!-- place globally-included scripts here -->


  

  <!-- opt-in code for integrating GraphQL with LIS Web Components -->
  <script type="importmap">
    {
      "imports": {
        "lis-graphql": "/assets/js/graphql/index.js",
        "lis-events": "/assets/js/events.js"
      }
    }
  </script>

  

  <!-- Icons -->
  <link rel="icon" type="image/png" sizes="16x16" href="/assets/icons/favicon-16x16.png">
      <link rel="icon" type="image/png" sizes="192x192" href="/assets/icons/favicon-192x192.png">
      <link rel="icon" href="/assets/icons/favicon.ico">
      </head>


<body>




<div id="navbar" class="uk-navbar-container tm-navbar-container uk-sticky uk-sticky-fixed uk-light" uk-sticky>

  <div id="alerts">
    
    
      

    
  </div>

  <script type="text/javascript">

    // define the cookie key
    const alertsKey = "lis-navbar-alerts";

    // get the alerts element
    const alerts = document.getElementById("alerts");

    // define a callback that saves the toggle state
    function toggleCallback(event) {
      document.cookie = `${alertsKey}=${event.type};SameSite=Lax;path=/`;
    }

    // use the callback every time the element is toggled
    alerts.addEventListener("show", toggleCallback);
    alerts.addEventListener("hide", toggleCallback);

    // load the element's state
    const alertsCookie = document.cookie
      .split('; ')
      .find(row => row.startsWith(`${alertsKey}=`));

    // hide the alerts if they're supposed to be hidden
    if (alertsCookie && alertsCookie.split('=')[1].toLowerCase() === "hide") {
      UIkit.toggle(alerts).toggle();
    }

  </script>

  <div class="uk-container uk-container-expand">
    <nav class="uk-navbar">
      <div class="uk-navbar-left uk-text-truncate">
        <a class="uk-navbar-item uk-logo uk-text-truncate" href="/">
          
          <img class="uk-margin-small-right" src="/assets/img/logo.png" alt="LIS: Legume Information System logo" width="47" height="47" />
          
          <div class="uk-text-truncate">
            LIS: Legume Information System
            
            <div class="uk-navbar-subtitle uk-text-truncate">Information about legume traits for crop improvement</div>
            
          </div>
        </a>
      </div>
      <div class="uk-navbar-right uk-flex-none">

        <ul class="uk-navbar-nav uk-visible@l">
  <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          
            <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
      </ul>
    </div>
    
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
          <li><a href="/tools/search/pangene.html">Pangene Lookup Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
      </ul>
    </div>
    
  </li>
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>

<a href="#off-screen-menu" class="uk-navbar-toggle uk-hidden@s" uk-navbar-toggle-icon uk-toggle></a>


        <ul class="uk-navbar-nav uk-visible@s">

        
          <li><a class="tm-no-underline" href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span></a></li>
        
        
        
          <li><a class="tm-no-underline" href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span></a></li>
        

        </ul>
        
        
      </div>
    </nav>
    <nav class="uk-navbar tm-lower-navbar">
      <div class="uk-navbar-left">

        <ul class="uk-navbar-nav uk-hidden@l uk-visible@s">
  <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          
            <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
      </ul>
    </div>
    
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
          <li><a href="/tools/search/pangene.html">Pangene Lookup Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
      </ul>
    </div>
    
  </li>
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>


      </div>
    </nav>
  </div>
</div>

<div id="off-screen-menu" uk-offcanvas="overlay: true">
  <div class="uk-offcanvas-bar uk-dark  uk-background-secondary">

    <ul class="uk-nav-default uk-nav-parent-icon" uk-nav>
  <li><a href="/">Home</a></li>
  
  <li class="uk-parent">
    <a href="#">Taxa</a>
    
    <ul class="uk-nav-sub">
      
        
          
            <li><a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li><a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li><a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li><a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li><a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li><a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li><a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li><a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
    </ul>
  </li>
  
  
  <li class="uk-parent">
    <a href="/tools">Tools</a>
    
    <ul class="uk-nav-sub">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
          <li><a href="/tools/search/pangene.html">Pangene Lookup Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
    </ul>
  </li>
  
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>


    
    <ul class="uk-nav uk-nav-default">
      <li class="uk-nav-divider"></li>
      
      <li><a href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span> Twitter</a></li>
      
      
      
      <li><a href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span> GitHub</a></li>
      
    </ul>
    

  </div>
</div>



  <div id="left-menu" class="tm-sidebar-left uk-visible@m">

  
  
  
    <h3>Data Collections</h3>

<ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
  
  
  
  <li><a href="/collections/arachis">Arachis</a></li>
  
  
  
  
  
  <li><a href="/collections/cajanus">Cajanus</a></li>
  
  
  
  
  
  <li><a href="/collections/cicer">Cicer</a></li>
  
  
  
  
  
  <li><a href="/collections/glycine">Glycine</a></li>
  
  
  
  
  
  <li><a href="/collections/lotus">Lotus</a></li>
  
  
  
  
  
  <li><a href="/collections/lupinus">Lupinus</a></li>
  
  
  
  
  
  <li><a href="/collections/medicago">Medicago</a></li>
  
  
  
  
  
  <li><a href="/collections/phaseolus">Phaseolus</a></li>
  
  
  
  
  
  <li><a href="/collections/pisum">Pisum</a></li>
  
  
  
  
  
  <li><a href="/collections/trifolium">Trifolium</a></li>
  
  
  
  
  
  <li><a href="/collections/vigna">Vigna</a></li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
</ul>

  

  <!-- dynamically set the top style property of the left menu when the height of the navbar changes -->
  <!-- NOTE: the script tag is nested so it doesn't break a CSS selector -->
  <script type="text/javascript">

    const navbar = document.getElementById("navbar");
    const left = document.getElementById("left-menu");

    const leftTopObserver = new ResizeObserver(entries => {
      for (let entry of entries) {
        // use the newer contentBoxSize property if the browser supports it
        if (entry.contentBoxSize) {
          left.style.top = entry.contentBoxSize[0].blockSize + "px";
        // fallback to the older contentRect property
        } else {
          left.style.top = entry.contentRect.height + "px";
        }
      }
    });

    leftTopObserver.observe(navbar);

  </script>
</div>


<!-- NOTE: putting content between the tools and tm-main blocks will break CSS selectors -->
<div class="tm-main uk-section uk-section-default">
  <div class="uk-container uk-flex">
    <div class="uk-width-1-1">
      


<h1><i>Glycine</i> data collections</h1>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/"><i>Glycine max</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/ZiHuaNo_4.gnm1.HN60/">ZiHuaNo_4.gnm1.HN60</a></dt>
  <dd>Genome assembly for Glycine max accession ZiHuaNo_4 (SoyL02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82.gnm5.NRKG/">Wm82.gnm5.NRKG</a></dt>
  <dd>Glycine max accession Williams 82 (Wm82), genome assembly 5</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/JinDouNo_23.gnm1.GBYD/">JinDouNo_23.gnm1.GBYD</a></dt>
  <dd>Genome assembly for Glycine max accession JinDouNo_23 (SoyC07) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/JuXuanNo_23.gnm1.WC57/">JuXuanNo_23.gnm1.WC57</a></dt>
  <dd>Genome assembly for Glycine max accession JuXuanNo_23 (SoyC03) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Lee.gnm1.BXNC/">Lee.gnm1.BXNC</a></dt>
  <dd>Genome assembly</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/TieJiaSiLiHuang.gnm1.W9PV/">TieJiaSiLiHuang.gnm1.W9PV</a></dt>
  <dd>Genome assembly for Glycine max accession TieJiaSiLiHuang (SoyL08) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/58-161.gnm1.BW8J/">58-161.gnm1.BW8J</a></dt>
  <dd>Genome assembly for Glycine max accession 58-161 (SoyL04) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/XuDouNo_1.gnm1.0GWH/">XuDouNo_1.gnm1.0GWH</a></dt>
  <dd>Genome assembly for Glycine max accession XuDouNo_1 (SoyC01) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/PI_548362.gnm1.V9S7/">PI_548362.gnm1.V9S7</a></dt>
  <dd>Genome assembly for Glycine max accession PI_548362 (SoyC10) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Lee.gnm3.VG1C/">Lee.gnm3.VG1C</a></dt>
  <dd>Glycine max accession Lee, genome assembly 3</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Zhutwinning2.gnm1.GR6N/">Zhutwinning2.gnm1.GR6N</a></dt>
  <dd>Genome assembly for Glycine max accession Zhutwinning2 (SoyL01) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/FengDiHuang.gnm1.YYLM/">FengDiHuang.gnm1.YYLM</a></dt>
  <dd>Genome assembly for Glycine max accession FengDiHuang (SoyL07) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82.gnm4.4PTR/">Wm82.gnm4.4PTR</a></dt>
  <dd>Glycine max accession Williams 82 genome assembly v4.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/JiDouNo_17.gnm1.MRYM/">JiDouNo_17.gnm1.MRYM</a></dt>
  <dd>Genome assembly for Glycine max accession JiDouNo_17 (SoyC11) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Zh13.gnm1.N6C8/">Zh13.gnm1.N6C8</a></dt>
  <dd>Genome assembly files for cultivar Zhonghuang 13, Shen et al. (2018)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/HanDouNo_5.gnm1.11MF/">HanDouNo_5.gnm1.11MF</a></dt>
  <dd>Genome assembly for Glycine max accession HanDouNo_5 (SoyC09) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/QiHuangNo_34.gnm1.MB9G/">QiHuangNo_34.gnm1.MB9G</a></dt>
  <dd>Genome assembly for Glycine max accession QiHuangNo_34 (SoyC08) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Huaxia3_IGA1007.gnm1.RGGN/">Huaxia3_IGA1007.gnm1.RGGN</a></dt>
  <dd>Genome assembly files for cultivar Huaxia3 (Huaxia3_IGA1007 in publication; WHFS_GmHX3_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Jinyuan_IGA1006.gnm1.LXM0/">Jinyuan_IGA1006.gnm1.LXM0</a></dt>
  <dd>Genome assembly files for cultivar Jinyuan (Jinyuan_IGA100 in the publication; 6HFS_GmJY_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/HeiHeNo_43.gnm1.FV4H/">HeiHeNo_43.gnm1.FV4H</a></dt>
  <dd>Genome assembly for Glycine max accession HeiHeNo_43 (SoyC13) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/ZhangChunManCangJin.gnm1.FC4Q/">ZhangChunManCangJin.gnm1.FC4Q</a></dt>
  <dd>Genome assembly for Glycine max accession ZhangChunManCangJin (SoyL06) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Zh13.gnm2.LV9P/">Zh13.gnm2.LV9P</a></dt>
  <dd>Genome assembly version 2 files for cultivar Zhonghuang 13, Shen et al. (2019)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/TieFengNo_18.gnm1.BW65/">TieFengNo_18.gnm1.BW65</a></dt>
  <dd>Genome assembly for Glycine max accession TieFengNo_18 (SoyC02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82_NJAU.gnm1.N4GV/">Wm82_NJAU.gnm1.N4GV</a></dt>
  <dd>Glycine max accession Williams 82 from Nanjing Agricultural University (Wm82-NJAU), genome assebly v1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/PI_398296.gnm1.D07C/">PI_398296.gnm1.D07C</a></dt>
  <dd>Genome assembly for Glycine max accession PI_398296 (SoyL05) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/WanDouNo_28.gnm1.ZRP2/">WanDouNo_28.gnm1.ZRP2</a></dt>
  <dd>Genome assembly for Glycine max accession WanDouNo_28 (SoyC04) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Zh35_IGA1004.gnm1.DBYJ/">Zh35_IGA1004.gnm1.DBYJ</a></dt>
  <dd>Genome assembly files for cultivar Zhonghuang 35 (Zh35_IGA1004 in publication; WHFS_GmZH35_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82_IGA1008.gnm1.5CQQ/">Wm82_IGA1008.gnm1.5CQQ</a></dt>
  <dd>Genome assembly files for cultivar Williams 82 (Wm82_IGA1008 in publication; WHFS_GmW82_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Amsoy.gnm1.TYYG/">Amsoy.gnm1.TYYG</a></dt>
  <dd>Genome assembly for Glycine max accession Amsoy (SoyC05) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82.gnm1.FCtY/">Wm82.gnm1.FCtY</a></dt>
  <dd>Glycine max accession Williams genome assembly v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/KeShanNo_1.gnm1.3TDD/">KeShanNo_1.gnm1.3TDD</a></dt>
  <dd>Genome assembly for Glycine max accession KeShanNo_1 (SoyC14) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Lee.gnm2.K7BV/">Lee.gnm2.K7BV</a></dt>
  <dd>Glycine max genotype Lee genome assembly v2.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82_ISU01.gnm2.JFPQ/">Wm82_ISU01.gnm2.JFPQ</a></dt>
  <dd>Glycine max accession Williams 82 ISU01, genome assembly v2</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/YuDouNo_22.gnm1.T2VD/">YuDouNo_22.gnm1.T2VD</a></dt>
  <dd>Genome assembly for Glycine max accession YuDouNo_22 (SoyC06) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wenfeng7_IGA1001.gnm1.L0QH/">Wenfeng7_IGA1001.gnm1.L0QH</a></dt>
  <dd>Genome assembly files for cultivar Wenfeng 7 (Wenfeng7_IGA1001 in publication; WHFS_GmWF7_1.0 in the GenBank assembly record); Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/JD17.gnm1.TW7L/">JD17.gnm1.TW7L</a></dt>
  <dd>Glycine max accession Jidou 17 (JD17), genome assembly 1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/TongShanTianEDan.gnm1.B4R6/">TongShanTianEDan.gnm1.B4R6</a></dt>
  <dd>Genome assembly for Glycine max accession TongShanTianEDan (SoyL03) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/FiskebyIII.gnm1.F177/">FiskebyIII.gnm1.F177</a></dt>
  <dd>Glycine max genotype Fiskeby III genome assembly v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/DongNongNo_50.gnm1.XRTN/">DongNongNo_50.gnm1.XRTN</a></dt>
  <dd>Genome assembly for Glycine max accession DongNongNo_50 (SoyC12) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Wm82.gnm2.DTC4/">Wm82.gnm2.DTC4</a></dt>
  <dd>Glycine max accession Williams 82 genome assembly v2.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Hwangkeum.gnm1.4S83/">Hwangkeum.gnm1.4S83</a></dt>
  <dd>Glycine max genotype Hwangkeum genome assembly v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Hefeng25_IGA1002.gnm1.L69T/">Hefeng25_IGA1002.gnm1.L69T</a></dt>
  <dd>Genome assembly files for cultivar Hefeng 25 (Hefeng25_IGA1002 in publication; WHFS_GmHF25_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/ShiShengChangYe.gnm1.MC01/">ShiShengChangYe.gnm1.MC01</a></dt>
  <dd>Genome assembly for Glycine max accession ShiShengChangYe (SoyL09) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/genomes/Zh13_IGA1005.gnm1.FRXQ/">Zh13_IGA1005.gnm1.FRXQ</a></dt>
  <dd>Genome assembly files for cultivar Zhonghuang 13 (Zh13_IGA1005 in publication; WHFS_GmZH13_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/TieFengNo_18.gnm1.ann1.7GR4/">TieFengNo_18.gnm1.ann1.7GR4</a></dt>
  <dd>Gene annotations for Glycine max accession TieFengNo_18 (SoyC02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/JinDouNo_23.gnm1.ann1.SGJW/">JinDouNo_23.gnm1.ann1.SGJW</a></dt>
  <dd>Gene annotations for Glycine max accession JinDouNo_23 (SoyC07) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Zh35_IGA1004.gnm1.ann1.RGN6/">Zh35_IGA1004.gnm1.ann1.RGN6</a></dt>
  <dd>Genome annotation files for cultivar Zhonghuang 35,  Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Zhutwinning2.gnm1.ann1.ZTTQ/">Zhutwinning2.gnm1.ann1.ZTTQ</a></dt>
  <dd>Gene annotations for Glycine max accession Zhutwinning2 (SoyL01) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/YuDouNo_22.gnm1.ann1.HCQ1/">YuDouNo_22.gnm1.ann1.HCQ1</a></dt>
  <dd>Gene annotations for Glycine max accession YuDouNo_22 (SoyC06) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82.gnm5.ann1.J7HW/">Wm82.gnm5.ann1.J7HW</a></dt>
  <dd>Annotation 1 for Glycine max accession Williams 82 (Wm82), genome assembly 5</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82.gnm4.ann1.T8TQ/">Wm82.gnm4.ann1.T8TQ</a></dt>
  <dd>Glycine max accession Williams genome annotation files; JGI annotation version 1 on assembly 4</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Hefeng25_IGA1002.gnm1.ann1.320V/">Hefeng25_IGA1002.gnm1.ann1.320V</a></dt>
  <dd>Genome annotation files for cultivar Hefeng 25, Chu et al (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/FengDiHuang.gnm1.ann1.P6HL/">FengDiHuang.gnm1.ann1.P6HL</a></dt>
  <dd>Gene annotations for Glycine max accession FengDiHuang (SoyL07) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/XuDouNo_1.gnm1.ann1.G2T7/">XuDouNo_1.gnm1.ann1.G2T7</a></dt>
  <dd>Gene annotations for Glycine max accession XuDouNo_1 (SoyC01) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/TieJiaSiLiHuang.gnm1.ann1.W70Z/">TieJiaSiLiHuang.gnm1.ann1.W70Z</a></dt>
  <dd>Gene annotations for Glycine max accession TieJiaSiLiHuang (SoyL08) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82_NJAU.gnm1.ann1.KM71/">Wm82_NJAU.gnm1.ann1.KM71</a></dt>
  <dd>Annotation 1 for Glycine max accession Williams 82 from Nanjing Agricultural University (Wm82-NJAU), genome assembly v1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/ZiHuaNo_4.gnm1.ann1.FCFQ/">ZiHuaNo_4.gnm1.ann1.FCFQ</a></dt>
  <dd>Gene annotations for Glycine max accession ZiHuaNo_4 (SoyL02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/ZhangChunManCangJin.gnm1.ann1.7HPB/">ZhangChunManCangJin.gnm1.ann1.7HPB</a></dt>
  <dd>Gene annotations for Glycine max accession ZhangChunManCangJin (SoyL06) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/QiHuangNo_34.gnm1.ann1.WHRV/">QiHuangNo_34.gnm1.ann1.WHRV</a></dt>
  <dd>Gene annotations for Glycine max accession QiHuangNo_34 (SoyC08) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/JD17.gnm1.ann1.CLFP/">JD17.gnm1.ann1.CLFP</a></dt>
  <dd>Annotation 1 for Glycine max accession Jidou 17 (JD17), genome assembly 1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/WanDouNo_28.gnm1.ann1.NLYP/">WanDouNo_28.gnm1.ann1.NLYP</a></dt>
  <dd>Gene annotations for Glycine max accession WanDouNo_28 (SoyC04) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Hwangkeum.gnm1.ann1.1G4F/">Hwangkeum.gnm1.ann1.1G4F</a></dt>
  <dd>Genome annotation files for Glycine max genotype Hwangkeum, Kim et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/FiskebyIII.gnm1.ann1.SS25/">FiskebyIII.gnm1.ann1.SS25</a></dt>
  <dd>Genome annotation files for Glycine max genotype Fiskeby III, Stupar et al. (2020)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Lee.gnm1.ann1.6NZV/">Lee.gnm1.ann1.6NZV</a></dt>
  <dd>Glycine max accession Lee genome annotation files; JGI annotation version 1 on assembly version 1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Amsoy.gnm1.ann1.6S5P/">Amsoy.gnm1.ann1.6S5P</a></dt>
  <dd>Gene annotations for Glycine max accession Amsoy (SoyC05) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82.gnm1.ann1.DvBy/">Wm82.gnm1.ann1.DvBy</a></dt>
  <dd>Glycine max accession Williams genome annotation files</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Zh13.gnm1.ann1.8VV3/">Zh13.gnm1.ann1.8VV3</a></dt>
  <dd>Genome annotation files for cultivar Zhonghuang 13, Shen et al. (2018)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Jinyuan_IGA1006.gnm1.ann1.2NNX/">Jinyuan_IGA1006.gnm1.ann1.2NNX</a></dt>
  <dd>Genome annotation files for cultivar Jinyuan, Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/58-161.gnm1.ann1.HJ1K/">58-161.gnm1.ann1.HJ1K</a></dt>
  <dd>Gene annotations for Glycine max accession 58-161 (SoyL04) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82.gnm2.ann1.RVB6/">Wm82.gnm2.ann1.RVB6</a></dt>
  <dd>Glycine max accession Williams genome annotation files; JGI annotation version 1 on assembly 2</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/JiDouNo_17.gnm1.ann1.X5PX/">JiDouNo_17.gnm1.ann1.X5PX</a></dt>
  <dd>Gene annotations for Glycine max accession JiDouNo_17 (SoyC11) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/PI_548362.gnm1.ann1.LL84/">PI_548362.gnm1.ann1.LL84</a></dt>
  <dd>Gene annotations for Glycine max accession PI_548362 (SoyC10) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/ShiShengChangYe.gnm1.ann1.VLGS/">ShiShengChangYe.gnm1.ann1.VLGS</a></dt>
  <dd>Gene annotations for Glycine max accession ShiShengChangYe (SoyL09) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/DongNongNo_50.gnm1.ann1.QSDB/">DongNongNo_50.gnm1.ann1.QSDB</a></dt>
  <dd>Gene annotations for Glycine max accession DongNongNo_50 (SoyC12) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Zh13_IGA1005.gnm1.ann1.87Z5/">Zh13_IGA1005.gnm1.ann1.87Z5</a></dt>
  <dd>Genome annotation files for cultivar Zhonghuang 13, Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Huaxia3_IGA1007.gnm1.ann1.LKC7/">Huaxia3_IGA1007.gnm1.ann1.LKC7</a></dt>
  <dd>Genome annotation files for cultivar Huaxia3, Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/KeShanNo_1.gnm1.ann1.2YX4/">KeShanNo_1.gnm1.ann1.2YX4</a></dt>
  <dd>Gene annotations for Glycine max accession KeShanNo_1 (SoyC14) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wenfeng7_IGA1001.gnm1.ann1.ZK5W/">Wenfeng7_IGA1001.gnm1.ann1.ZK5W</a></dt>
  <dd>Genome annotation files for cultivar  Wenfeng 7, Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/TongShanTianEDan.gnm1.ann1.56XW/">TongShanTianEDan.gnm1.ann1.56XW</a></dt>
  <dd>Gene annotations for Glycine max accession TongShanTianEDan (SoyL03) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/HeiHeNo_43.gnm1.ann1.PDXG/">HeiHeNo_43.gnm1.ann1.PDXG</a></dt>
  <dd>Gene annotations for Glycine max accession HeiHeNo_43 (SoyC13) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Zh13.gnm2.ann1.FJ3G/">Zh13.gnm2.ann1.FJ3G</a></dt>
  <dd>Genome annotation files for cultivar Zhonghuang 13, Shen et al. (2019)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/PI_398296.gnm1.ann1.B0XR/">PI_398296.gnm1.ann1.B0XR</a></dt>
  <dd>Gene annotations for Glycine max accession PI_398296 (SoyL05) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/HanDouNo_5.gnm1.ann1.ZS7M/">HanDouNo_5.gnm1.ann1.ZS7M</a></dt>
  <dd>Gene annotations for Glycine max accession HanDouNo_5 (SoyC09) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82_ISU01.gnm2.ann1.FGFB/">Wm82_ISU01.gnm2.ann1.FGFB</a></dt>
  <dd>Annotation 1 for Glycine max accession Williams 82 ISU01, genome assembly v2</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Lee.gnm2.ann1.1FNT/">Lee.gnm2.ann1.1FNT</a></dt>
  <dd>genome annotation of Glycine max Lee</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Wm82_IGA1008.gnm1.ann1.FGN6/">Wm82_IGA1008.gnm1.ann1.FGN6</a></dt>
  <dd>Genome annotation files for cultivar William 82, Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/Lee.gnm3.ann1.ZYY3/">Lee.gnm3.ann1.ZYY3</a></dt>
  <dd>Annotation 1 for Glycine max accession Lee, genome assembly 3</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/annotations/JuXuanNo_23.gnm1.ann1.H8PW/">JuXuanNo_23.gnm1.ann1.H8PW</a></dt>
  <dd>Gene annotations for Glycine max accession JuXuanNo_23 (SoyC03) from Liu, Du et al. 2020.</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/expression/">Expression</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/expression/Wm82.gnm2.ann1.expr.Wm82.Libault_Farmer_2010/">Wm82.gnm2.ann1.expr.Wm82.Libault_Farmer_2010</a></dt>
  <dd>Soybean gene expression atlas from 14 conditions (Libault et al., 2010).</dd>
  
</dl>









<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/">Markers</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm4.mrk.SoySNP50K/">Wm82.gnm4.mrk.SoySNP50K</a></dt>
  <dd>SoySNP50K is an Illumina Infinium II BeadChip that contains over 50,000 SNPs from soybean useful for genotyping a wide variety of cultivated and wild soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.BARCSOYSSR/">Wm82.gnm1.mrk.BARCSOYSSR</a></dt>
  <dd>BARCSOYSSR contains 33,065 locus-specific SSR markers with di-, tri-, and tetranucleotide repeats of five or more.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.SoySSR/">Wm82.gnm2.mrk.SoySSR</a></dt>
  <dd>This marker set contains SSRs developed and published by Cregan, et al. in 1999, with names starting with Satt or Sat_.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Sonah_ODonoughue_2015/">Wm82.gnm2.mrk.Sonah_ODonoughue_2015</a></dt>
  <dd>GWAS study of eight traits in soybean, conducted on 139 accessions, genotyped using GBS-derived markers.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.Zhao_Teng_2017/">Wm82.gnm1.mrk.Zhao_Teng_2017</a></dt>
  <dd>SLAF-seq markers associated with resistance to soybean cyst nematode (SCN) HG type 2.5.7, with respect to Wm82 gnm1 coordinates.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.SoySNP6K/">Wm82.gnm2.mrk.SoySNP6K</a></dt>
  <dd>Illumina Infinium SoySNP6K BeadChip</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82_ISU01.gnm2.mrk.SoySNP50K/">Wm82_ISU01.gnm2.mrk.SoySNP50K</a></dt>
  <dd>SoySNP50K is an Illumina Infinium II BeadChip that contains over 50,000 SNPs from soybean useful for genotyping a wide variety of cultivated and wild soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Zhao_Teng_2017/">Wm82.gnm2.mrk.Zhao_Teng_2017</a></dt>
  <dd>SLAF-seq markers associated with resistance to soybean cyst nematode (SCN) HG type 2.5.7, with respect to Wm82 gnm2 coordinates (derived via sequence homology, from markers originally with positions in Wm82 gnm1).</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Fang_Ma_2017/">Wm82.gnm2.mrk.Fang_Ma_2017</a></dt>
  <dd>GWAS of 84 agronomic traits, conducted on 809 soybean accessions, with genotyping by resequencing.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.BARCSOYSSR/">Wm82.gnm2.mrk.BARCSOYSSR</a></dt>
  <dd>BARCSOYSSR contains 33,065 locus-specific SSR markers with di-, tri-, and tetranucleotide repeats of five or more.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Tran_Steketee_2019/">Wm82.gnm2.mrk.Tran_Steketee_2019</a></dt>
  <dd>Diverse soybean accessions (462) were screened for resistance to SCN and genotyped using SoySNP50K Infinium Chip and a set of 3 funtional KASP SNP markers, identifying both 88788-type and Peking-type resistance.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Li_Zhao_2019/">Wm82.gnm2.mrk.Li_Zhao_2019</a></dt>
  <dd>12,072 SNPs from Li, Zhao, et al. (2019), a seed protein and oil study.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/FiskebyIII.gnm1.mrk.SoySNP50K/">FiskebyIII.gnm1.mrk.SoySNP50K</a></dt>
  <dd>SoySNP50K is an Illumina Infinium II BeadChip that contains over 50,000 SNPs from soybean useful for genotyping a wide variety of cultivated and wild soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.SoySNP6K/">Wm82.gnm1.mrk.SoySNP6K</a></dt>
  <dd>Illumina Infinium SoySNP6K BeadChip</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.SoySSR/">Wm82.gnm1.mrk.SoySSR</a></dt>
  <dd>This marker set contains SSRs developed and published by Cregan, et al. in 1999, with names starting with Satt or Sat_.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Han_Zhao_2015/">Wm82.gnm2.mrk.Han_Zhao_2015</a></dt>
  <dd>SNP markers generated Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach, and showing association with resistance against two soybean cyst nematode types.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Vuong_Sonah_2015/">Wm82.gnm2.mrk.Vuong_Sonah_2015</a></dt>
  <dd>Markers associated with Soybean Cyst Nematode (SCN) resistance in Soybean, relative to the Wm82.gnm2 assembly. Markers derive from the SoySNP50K set.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.SoySNP50K/">Wm82.gnm1.mrk.SoySNP50K</a></dt>
  <dd>SoySNP50K is an Illumina Infinium II BeadChip that contains over 50,000 SNPs from soybean useful for genotyping a wide variety of cultivated and wild soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.1536_USLP1/">Wm82.gnm2.mrk.1536_USLP1</a></dt>
  <dd>Universal Soy Linkage Panel (USLP 1.0) contains 1536 SNPs selected based on even distribution throughout each of the 20 consensus linkage groups and to have a broad range of allele frequencies in diverse germplasm.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.NJAU355K/">Wm82.gnm2.mrk.NJAU355K</a></dt>
  <dd>NJAU 355K SoySNP array used to study soybean domestication in Wang, et al. (2016).</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.SoySNP50K/">Wm82.gnm2.mrk.SoySNP50K</a></dt>
  <dd>SoySNP50K is an Illumina Infinium II BeadChip that contains over 50,000 SNPs from soybean useful for genotyping a wide variety of cultivated and wild soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm1.mrk.Li_Guo_2016/">Wm82.gnm1.mrk.Li_Guo_2016</a></dt>
  <dd>59,845 single-nucleotide polymorphisms identified from re-sequencing of 279 accessions from Yangtze-Huai soybean breeding germplasm.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.Meng_He_2016/">Wm82.gnm2.mrk.Meng_He_2016</a></dt>
  <dd>GWAS study of seed isoflavone content (SIFC), conducted on 366 landraces, genotyped with 116,769 RAD-Seq markers.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/markers/Wm82.gnm2.mrk.SoyaSNP180K/">Wm82.gnm2.mrk.SoyaSNP180K</a></dt>
  <dd>Axiom SoyaSNP180K array with 180,981 SNPs developed by Lee, et al. (2015).</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/synteny/">Synteny</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/synteny/Wm82.gnm2.syn.HXNY/">Wm82.gnm2.syn.HXNY</a></dt>
  <dd>Synteny mappings among the genomes. See genome assemblies indicated in the synteny comparisons.</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/">Diversity</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Song_NAM_2021a/">Wm82.gnm2.div.Song_NAM_2021a</a></dt>
  <dd>Imputed genotypes for 5176 Nested Association Mapping RILs relative to 40 NAM parental genotypes, genotyped using the SoySNP50K and SoyNAM6K assays. These data sets report alleles at 29,417 SNP positions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Song_Wei_2015/">Wm82.gnm1.div.Song_Wei_2015</a></dt>
  <dd>The polymorphism information of INDELS and SNPs between Hedou12 and Wm82.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.dosSantos_Valliyodan_2016/">Wm82.gnm2.div.dosSantos_Valliyodan_2016</a></dt>
  <dd>VCF and HapMap file containing genotype information for 28 Brazilian soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Wickland_Battu_2017/">Wm82.gnm2.div.Wickland_Battu_2017</a></dt>
  <dd>VCF files containing genotype information for 3 populations of soybean produced by GB-eaSy.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Zhang_Jiang_2020/">Wm82.gnm2.div.Zhang_Jiang_2020</a></dt>
  <dd>Variant information for 1,556 resequenced soybean accessions Glycine max and Glycine soja, provided by Yong-Qiang (Charles) An and Rick Meyers, relative to Wm82 assembly 2.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Song_Hyten_2015/">Wm82.gnm2.div.Song_Hyten_2015</a></dt>
  <dd>VCF, HapMap, and Flapjack files containing genotype information of the whole U.S. soybean germplasm collection using the SoySNP50K BeadChip.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Song_Hyten_2015/">Wm82.gnm1.div.Song_Hyten_2015</a></dt>
  <dd>VCF, HapMap, and Flapjack files containing genotype information of the whole U.S. soybean germplasm collection using the SoySNP50K BeadChip.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Jeong_Moon_2019/">Wm82.gnm2.div.Jeong_Moon_2019</a></dt>
  <dd>Variant information for 4234 soybean accessions (Glycine max and Glycine soja) genotyped with the Axiom SoyaSNP array containing 180,961 SNPs.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm4.div.Song_Hyten_2015/">Wm82.gnm4.div.Song_Hyten_2015</a></dt>
  <dd>VCF and HapMap files containing genotype information of the whole U.S. soybean germplasm collection using the SoySNP50K BeadChip.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Wei_Mesquita_2017/">Wm82.gnm2.div.Wei_Mesquita_2017</a></dt>
  <dd>VCF file containing GBS information for 374 U.S. and Brazilian accessions against 68301 SNPs.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Valliyodan_Brown_2021/">Wm82.gnm2.div.Valliyodan_Brown_2021</a></dt>
  <dd>Variant information for 481 soybean accessions (Glycine max and Glycine soja), provided by Babu Valliyodan and Henry Nguyen, as part of the United Soybean Board project to genotype and characterize diverse accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Song_NAM_2021b/">Wm82.gnm2.div.Song_NAM_2021b</a></dt>
  <dd>Imputed genotypes for 5176 Nested Association Mapping RILs relative to 40 NAM parental genotypes, genotyped with WGS and the SoySNP50K and SoyNAM6K assays. These data sets report alleles at 423,419 SNP positions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.ContrerasSoto_Mora_2017/">Wm82.gnm1.div.ContrerasSoto_Mora_2017</a></dt>
  <dd>Haplotype information for 169 Brazilian soybean cultivars genotyped with the 6K SNPchip.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Song_Yan_2017/">Wm82.gnm2.div.Song_Yan_2017</a></dt>
  <dd>Haplotypes of SoyNAM parents and progeny with respect to Williams 82 gnm2.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Hu_Zhang_2020/">Wm82.gnm1.div.Hu_Zhang_2020</a></dt>
  <dd>Variant information for 96 Glycine soja accessions, genotyped using the NJAU 335K SoySNP array, provided by Guizhen Kan.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Lee_Jeong_2015/">Wm82.gnm2.div.Lee_Jeong_2015</a></dt>
  <dd>Text, VCF, HapMap, and Flapjack file containing genotype information for 222 Korean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Lam_Xu_2010/">Wm82.gnm2.div.Lam_Xu_2010</a></dt>
  <dd>VCF file containing genotype information for 31 wild and cultivated soybeans received from Meng Ni, Tin Hang, and Hon-Ming Lam.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Torkamaneh_Laroche_2019/">Wm82.gnm2.div.Torkamaneh_Laroche_2019</a></dt>
  <dd>HapMap Data, GmHapMap, for 1007 accession of Glycine max and Glycine soja, from Torkamaneh et al. (2019)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm2.div.Torkamaneh_Laroche_2017/">Wm82.gnm2.div.Torkamaneh_Laroche_2017</a></dt>
  <dd>Resequencing information for 102 Canadian soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Wang_Jiao_2016/">Wm82.gnm1.div.Wang_Jiao_2016</a></dt>
  <dd>Variant information in vcf format for 367 Chinese soybean accessions using the NJAU 355K SoySNP array, provided by Dr. Jiao Wang.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Zhou_Jiang_2015/">Wm82.gnm1.div.Zhou_Jiang_2015</a></dt>
  <dd>Resequencing information for 302 wild and cultivated soybean accessions.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/max/diversity/Wm82.gnm1.div.Song_Yan_2017/">Wm82.gnm1.div.Song_Yan_2017</a></dt>
  <dd>Haplotypes of SoyNAM parents and progeny with respect to Williams 82 genome assembly gnm1.</dd>
  
</dl>


<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/"><i>Glycine soja</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/PI_562565.gnm1.Z2VG/">PI_562565.gnm1.Z2VG</a></dt>
  <dd>Genome assembly for Glycine soja accession PI_562565 (SoyW01) from the 26 accession pan-genome set of Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/PI483463.gnm1.YJWS/">PI483463.gnm1.YJWS</a></dt>
  <dd>Glycine soja accession PI 483463 genome assembly, v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/F_IGA1003.gnm1.V9RB/">F_IGA1003.gnm1.V9RB</a></dt>
  <dd>Genome assembly files for Glycine soja F (F_IGA1003 in publication; WHFS_GsojaF_1.0 in the GenBank assembly record), Chu et al. (2021)</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/PI_578357.gnm1.1XT1/">PI_578357.gnm1.1XT1</a></dt>
  <dd>Genome assembly for Glycine soja accession PI_578357 (SoyW03) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/PI_549046.gnm1.TSLF/">PI_549046.gnm1.TSLF</a></dt>
  <dd>Genome assembly for Glycine soja accession PI_549046 (SoyW02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/genomes/W05.gnm1.SVL1/">W05.gnm1.SVL1</a></dt>
  <dd>Genome assembly files for cultivar W05 from Xie, Lam et al. (2019): A reference-grade wild soybean genome</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/W05.gnm1.ann1.T47J/">W05.gnm1.ann1.T47J</a></dt>
  <dd>Glycine soja accession W05 genome annotation files</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/PI_578357.gnm1.ann1.0ZKP/">PI_578357.gnm1.ann1.0ZKP</a></dt>
  <dd>Gene annotations for Glycine soja accession PI_578357 (SoyW03) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/PI_549046.gnm1.ann1.65KD/">PI_549046.gnm1.ann1.65KD</a></dt>
  <dd>Gene annotations for Glycine soja accession PI_549046 (SoyW02) from Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/PI483463.gnm1.ann1.3Q3Q/">PI483463.gnm1.ann1.3Q3Q</a></dt>
  <dd>Glycine soja accession PI483463 genome annotation files; JGI annotation version 1 on assembly 1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/PI_562565.gnm1.ann1.1JD2/">PI_562565.gnm1.ann1.1JD2</a></dt>
  <dd>Gene annotations for Glycine soja accession PI_562565 (SoyW01) from the 26 accession pan-genome set of Liu, Du et al. 2020.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/soja/annotations/F_IGA1003.gnm1.ann1.G61B/">F_IGA1003.gnm1.ann1.G61B</a></dt>
  <dd>Genome annotation files for cultivar F, Chu et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/cyrtoloba/"><i>Glycine cyrtoloba</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/cyrtoloba/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/cyrtoloba/genomes/G1267.gnm1.YWW6/">G1267.gnm1.YWW6</a></dt>
  <dd>Genome assemblies for Glycine cyrtoloba, accession G1267, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/cyrtoloba/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/cyrtoloba/annotations/G1267.gnm1.ann1.HRFD/">G1267.gnm1.ann1.HRFD</a></dt>
  <dd>Genome annotation files for Glycine cyrtoloba, accession G1267, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/dolichocarpa/"><i>Glycine dolichocarpa</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/dolichocarpa/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/dolichocarpa/genomes/G1134.gnm1.PP7B/">G1134.gnm1.PP7B</a></dt>
  <dd>Genome assemblies for Glycine dolichocarpa, accession G1134, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/dolichocarpa/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/dolichocarpa/annotations/G1134.gnm1.ann1.4BJM/">G1134.gnm1.ann1.4BJM</a></dt>
  <dd>Genome annotation files for Glycine dolichocarpa, accession G1134, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/falcata/"><i>Glycine falcata</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/falcata/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/falcata/genomes/G1718.gnm1.B1PY/">G1718.gnm1.B1PY</a></dt>
  <dd>Genome assemblies for Glycine falcata, accession G1718, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/falcata/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/falcata/annotations/G1718.gnm1.ann1.2KSV/">G1718.gnm1.ann1.2KSV</a></dt>
  <dd>Genome annotation files for Glycine falcata, accession G1718, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/stenophita/"><i>Glycine stenophita</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/stenophita/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/stenophita/genomes/G1974.gnm1.7MZB/">G1974.gnm1.7MZB</a></dt>
  <dd>Genome assemblies for Glycine stenophita, accession G1974, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/stenophita/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/stenophita/annotations/G1974.gnm1.ann1.F257/">G1974.gnm1.ann1.F257</a></dt>
  <dd>Genome annotation files for Glycine stenphyta, accession G1974, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/syndetika/"><i>Glycine syndetika</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/syndetika/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/syndetika/genomes/G1300.gnm1.C11H/">G1300.gnm1.C11H</a></dt>
  <dd>Genome assemblies for Glycine syndetika, accession G1300, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/syndetika/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/syndetika/annotations/G1300.gnm1.ann1.RRK6/">G1300.gnm1.ann1.RRK6</a></dt>
  <dd>Genome annotation files for Glycine syndetika, accession G1300, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Glycine/D3-tomentella/"><i>Glycine D3-tomentella</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/D3-tomentella/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/D3-tomentella/genomes/G1403.gnm1.CL6K/">G1403.gnm1.CL6K</a></dt>
  <dd>Genome assemblies for Glycine D3 tomentella, accession G1403, from Zhuang, Wang et al. (2021)</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Glycine/D3-tomentella/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Glycine/D3-tomentella/annotations/G1403.gnm1.ann1.XNZQ/">G1403.gnm1.ann1.XNZQ</a></dt>
  <dd>Genome annotation files for Glycine D3 tomentella, accession G1403, from Zhuang, Wang et al. (2021)</dd>
  
</dl>
















<hr/>




    </div>
    
  </div>
</div>


</body>
</html>
