---
title: FAIR Access to Data
tools_menu: false
---
<h2>Fair Practices</h2>
<p>“FAIR” is the acronym for <b>F</b>indable, <b>A</b>ccessible, <b>I</b>nteroperable and <b>R</b>eusable, and applies to scientific data used by people and computers. The FAIR data principles were established in 2016.  More information is available from the <a href="https://www.nature.com/articles/sdata201618" target="_blank">2016 publication</a>, the <a href="https://doi.org/10.1016/j.molp.2018.07.005" target="_blank">2018 editorial publication</a>, the <a href="https://force11.org/info/the-fair-data-principles/" target="_blank">FAIR website</a> and the <a href="https://www.go-fair.org/go-fair-initiative/" target="_blank">GO FAIR Initiative webpage</a>.</p>

<p class="uk-text-bold"><strong>The FAIR principles recognize that:</strong></p>
<ul class="uk-list uk-list-disc">
<li>published data should be stored where it can be found and identified accurately using globally unique persistent identifiers;</li>
<li>data can be retrieved using standard protocols (like through a web interface);</li>
<li>data is in a standard format such that it can be used in other applications; and</li>
<li>is licensed properly to allow reuse.</li>
</ul>

<p>At SoyBase, we value data and believe that it is the responsibility of every soybean researcher and breeder to make their publicly-funded data FAIR. Here we outline some basic guidelines for good data management. We hope that our breeding and research community will lead the world wide charge to FAIR data! We are always happy to answer your questions on these issues.</p>


<p class="uk-text-bold"><strong>Why is this important?</strong></p> 
<p>Published data continues to increase dramatically in volume and complexity. Data sets in individual publications are now routinely so large, that the aid of computer analysis is required. For data to be “machine readable” (that is, the data can be manipulated by a computer program), new standards are being established so that data is BOTH human and machine readable. Also, journal publishers often do not accept large data sets, so other data repositories must be used, and it can be difficult for submitters to find the right database for their data. It can also be difficult for researchers to find data associated with a publication that is not in the supplementary data, if persistent globally unique identifiers are not specified in the published article.</p>


<p class="uk-text-bold"><strong>Why should YOUR Data be FAIR?</strong></p>  
If you make your data FAIR, it will be more visible, easier to reuse, and more frequently cited. If others make their data FAIR, it will help the entire community harvest the vast depth and breath of soybean data, and discovery will proceed more quickly, especially as new analysis methods come online.


<p class="uk-text-bold"><strong>What can <i>YOU</i> do to make your data FAIR?</strong></p>

<p>Start with these tasks:</p>

<ul> 
	<li><b>Understand the FAIR data principles, and make your data FAIR</b></li>
		<ul class="uk-list uk-list-circle">
			<li>Start by reading <a href="https://www.nature.com/articles/sdata201618" target="_blank">FAIR Guiding Principles for scientific data</a>. It is also important to budget time for data management, just as you budget time for the other aspects of your research. </li>
		</ul>

	<li><b>Put your data in a permanent and stable database</b></li>
		<ul class="uk-list uk-list-circle">
			<li>There are permanent and stable repositories for many types of scientific data. Data should go into the correct repository, and <b><u>then it can be pulled into SoyBase for further curation and use with our tools</u></b>. Where ever you deposit data, get a DOI (or other persistent, globally unique identifier) and put it in your publication. Not sure where to put your data? Nature provides an excellent <a href="https://www.nature.com/sdata/policies/repositories" target="_blank">list</a> of data repositories and recommendations. The <a href="https://www.re3data.org/" target="_blank">re3data.org</a> and <a href="https://fairsharing.org/" target="_blank">FAIRsharing.org</a> websites have extensive lists of databases, resources, and repositories. If you are still unsure where to submit data, or need help submitting, please ask anyone at SoyBase. If your journal article refers to data NOT published with your article, please make sure to obtain and add a persistent identifier and location of your data in your article.</li>
			<li><u>Genome Assemblies:</u> Submit genome assemblies to <a href="https://www.ebi.ac.uk/submission/" target="_blank">EBI</a> or <a href="https://submit.ncbi.nlm.nih.gov/" target="_blank">NCBI Genomes</a>. We understand this can take some time to complete. <i>We can help, so please do not be tempted to simply submit contigs to Genbank.</i></li>
			<li><u>DNA/RNA/Protein Sequences:</u> All DNA,RNA and protein sequences need to be submitted to <a href="https://submit.ncbi.nlm.nih.gov/" target="_blank">NCBI</a>, <a href="https://www.ebi.ac.uk/submission/" target="_blank">EBI</a>, or <a href="https://www.ddbj.nig.ac.jp/ddbj/web-submission-e.html" target="_blank">DDBJ</a>. This databases provide a stable, long-term storage for DNA, RNA and protein sequence data and create stable identifiers for datasets. These three organizations share sequence data on a daily basis, so data deposited at one is available at all three.</li>
			<li><u>SNPs:</u> All soybean SNPs should be submitted to <a href="https://www.ebi.ac.uk/eva/?Submit-Data" target="_blank">EVA at EBI</a>.</li>
			<li><u>Gene Expression:</u> Data used in gene expression studies should be submitted to the <a href="https://www.ncbi.nlm.nih.gov/geo/info/submission.html" target="_blank">NCBI GEO</a>.</li>
			<li><u>Protein/Proteomics/Metabolomics:</u> Explore <a href="https://www.uniprot.org/" target="_blank">Uniprot</a>, <a href="https://www.ebi.ac.uk/metabolights/" target="_blank">MetaboLights</a>, <a href="https://peptideatlas.org/" target="_blank">Peptide Atlas</a>, and <a href="https://www.ebi.ac.uk/pride/" target="_blank">PRIDE</a>. Metabolomics Data should be submitted following the <a href="https://link.springer.com/article/10.1007/s11306-007-0070-6" target="_blank">MSI guidelines</a>. Submit proteomics data to members of the <a href="https://www.proteomexchange.org/" target="_blank">ProteomeXchange</a>, following the <a href="https://www.psidev.info/miape" target="_blank">MIAPE recommendations</a>.</li>
			<li><u>General Repositories:</u> <a href="https://datadryad.org/stash" target="_blank">Data Dryad</a> or <a href="https://figshare.com/" target="_blank">Figshare</a></li>
		</ul>

	<li><b>Understand “Machine Readable”</b></li>
		<ul class="uk-list uk-list-circle">
			<li>This simply means that data is in a format that can be read and processed by a computer without human intervention. Computers are good at exact matches; for example, "lg1" does NOT equal "liguleless1" and “Chr1” does NOT equal “1” to a computer. Word documents and pdfs are NOT machine readable. Formats such as spreadsheets with header columns that can be exported as comma separated values (CSV), or standard formats for specific data types like FASTA, FASTQ, BED, GFF3, BAM, SAM, VCF, etc., ARE machine readable. Repositories often describe the machine-readable file formats they accept. If you have questions about what file formats SoyBase accepts please contact the SoyBase team.</li>
		</ul>
	
	<li><b>Attach complete and detailed metadata to your data sets, and use accepted file formats</b></li>
		<ul class="uk-list uk-list-circle">
			<li>When you deposit data, you are asked for information about your data (metadata). Please give this the same careful attention you give to your bench work and analysis. Datasets that are not adequately described are not reusable or reproducible, and raise questions about the carefulness and accuracy of the research. You should supply enough metadata so that your experiment can be reproduced. Be sure to use community standards for your datatype, such as <a href="https://www.nature.com/articles/nbt.1823" tareget="_blank">MIxS</a> (Minimal Information about any Sequence) or <a href="https://www.miappe.org/" target="_blank">MIAPPE</a> (Minimum Information About a Plant Phenotyping Experiment). These standards will inform you on what information to provide and the accepted file formats for your type of data. Standards can also be found at data repositories.</li>
			<li>Use ontology terms to describe your data. <a href="https://www.nature.com/scitable/topicpage/ontologies-scientific-data-sharing-made-easy-77972/" target="_blank">Ontologies</a> provide a powerful organizing framework for data, and help data to be machine readable.</li>
		</ul>
		
	<li><b>Do not rename genes that already have names</b></li>
		<ul class="uk-list uk-list-circle">
			<li>Once upon a time, the name of a soybean gene was its unique, persistent identifier. But now, renaming of genes that already have names is a big problem. Many names for the same genes make it difficult to find all information for that gene. Even worse, when the same name is used for different genes, how can a human, much less a computer know they are different genes? Please look up your gene at SoyBase before assigning a name, and follow the <a href="/about/" target="_blank">SoyBase nomenclature guidelines.</a></li>
		</ul>

	<li><a href="/about/" target="_blank"><b>Let SoyBase know about your work!</b></a></li>
		<ul class="uk-list uk-list-circle">
			<li>Please let us know about your publications, and provide links to your data. If you have published on a gene or genes, or if you have a dataset that will be useful to others, let us know. Review the  pages on genes or other information that you study, and let us know if corrections should be made. We want to get it right!</li>
		</ul>
</ul>
