<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Phaseolus &#8211; LIS: Legume Information System</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="A collaborative, community resource to facilitate crop improvement by integrating genetic, genomic, and trait data across legume species.">
  <link rel="manifest" type="application/manifest+json; charset=utf-8" href="/manifest.json" />
  <meta name="robots" content="all">
  <meta name="author" content="">
  
  
  <link rel="canonical" href="/collections/phaseolus/">
  <link rel="alternate" type="application/rss+xml" title="RSS Feed for LIS: Legume Information System" href="/feed.xml" />
  
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-49655154-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-49655154-1');
</script>

  
  <!-- Custom CSS -->
  <link rel="stylesheet" href="/assets/css/uikit.min.css" type="text/css" />
  <link rel="stylesheet" href="/assets/css/uikit-theme.css" type="text/css" />
  
  <!-- place globally-included stylesheet content here -->
<style>
  .species-image {
      float: right;
      margin: 10px;
  }
  .species-image img {
      border: 1px solid darkgreen;
  }
  .species-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .blog-image {
      float: right;
      margin: 10px;
  }
  .blog-image img {
      border: 1px solid gray;
  }
  .blog-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .temp {
      color: darkred;
      font-family: serif;
      font-weight: bold;
  }
</style>



  <!-- Custom JS -->
  <script src="/assets/js/uikit.min.js"></script>
  <script src="/assets/js/uikit-icons.min.js"></script>

  <!-- place globally-included scripts here -->

<!-- experimenting with the JS compiled under jekyll-data-js -->
<script src="/assets/js/lis-data.js"></script>



  <!-- Icons -->
  <link rel="icon" type="image/png" href="/assets/icons/favicon-192x192.png" sizes="192x192">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-16x16.png" sizes="16x16">
  <link rel="shortcut icon" href="/assets/icons/favicon.ico">
</head>


<body>




<div id="navbar" class="uk-navbar-container tm-navbar-container uk-sticky uk-sticky-fixed uk-light" uk-sticky>

  <div id="alerts">
    
    
      <div class="uk-alert-primary uk-margin-remove" uk-alert>
  <p><b>Welcome to the new LIS website!</b> The old website can be found at <a href="https://legacy.legumeinfo.org/" target="_blank">legacy.legumeinfo.org</a>. Click the bell (<span uk-icon="bell"></span>) in the navigation bar to toggle this alert.</p>
</div>

    
  </div>

  <script type="text/javascript">

    // define the cookie key
    const alertsKey = "lis-navbar-alerts";

    // get the alerts element
    const alerts = document.getElementById("alerts");

    // define a callback that saves the toggle state
    function toggleCallback(event) {
      document.cookie = `${alertsKey}=${event.type};SameSite=Lax;path=/`;
    }

    // use the callback every time the element is toggled
    alerts.addEventListener("show", toggleCallback);
    alerts.addEventListener("hide", toggleCallback);

    // load the element's state
    const alertsCookie = document.cookie
      .split('; ')
      .find(row => row.startsWith(`${alertsKey}=`));

    // hide the alerts if they're supposed to be hidden
    if (alertsCookie && alertsCookie.split('=')[1].toLowerCase() === "hide") {
      UIkit.toggle(alerts).toggle();
    }

  </script>

  <div class="uk-container uk-container-expand">
    <nav class="uk-navbar">
      <div class="uk-navbar-left uk-text-truncate">
        <a class="uk-navbar-item uk-logo uk-text-truncate" href="/">
          
          <img class="uk-margin-small-right" src="/assets/img/logo.png" width="47" height="47" />
          
          <div class="uk-text-truncate">
            LIS: Legume Information System
            
            <div class="uk-navbar-subtitle uk-text-truncate">Information about legume traits for crop improvement</div>
            
          </div>
        </a>
      </div>
      <div class="uk-navbar-right uk-flex-none">

        <ul class="uk-navbar-nav uk-visible@l">
    <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        
        <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <li class="uk-nav-divider"></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <li><a href="/taxa/aeschynomene"> Aeschynomene (Jointvetch)</a></li>
        
        
        
        
        
        <li><a href="/taxa/apios"> Apios (Potato bean)</a></li>
        
        
        
        
        
        <li><a href="/taxa/bauhinia"> Bauhinia (Orchid tree)</a></li>
        
        
        
        
        
        <li><a href="/taxa/cercis"> Cercis (Redbud tree)</a></li>
        
        
        
        
        
        <li><a href="/taxa/chamaecrista"> Chamaecrista (Partridge pea)</a></li>
        
        
        
        
        
        <li><a href="/taxa/faidherbia"> Faidherbia (Apple-ring acacia)</a></li>
        
        
        
        
        
        <li><a href="/taxa/zenia"> Zenia (Southeast Asian tree)</a></li>
        
        
        
      </ul>
    </div>
  </li>
  <li>
    <a href="/collections">Collections</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        
        <li> <a href="/collections/arachis"> Arachis (Peanut)</a></li>
        
        
        
        
        
        <li> <a href="/collections/cajanus"> Cajanus (Pigeonpea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/cicer"> Cicer (Chickpea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/glycine"> Glycine (Soybean)</a></li>
        
        
        
        
        
        <li> <a href="/collections/lotus"> Lotus (Bird's-foot trefoil)</a></li>
        
        
        
        
        
        <li> <a href="/collections/lupinus"> Lupinus (Lupin grain legumes)</a></li>
        
        
        
        
        
        <li> <a href="/collections/medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
        
        
        
        
        
        <li> <a href="/collections/phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
        
        
        
        
        
        <li> <a href="/collections/pisum"> Pisum (Pea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/trifolium"> Trifolium (Clover)</a></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
      </ul>
    </div>
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
        <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
        <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
        <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
        <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
        <li><a href="https://cmap.legumeinfo.org/">CMap</a></li>
        
        <li><a href="https://legumeinfo.org/germplasm/map">LIS Germplasm GIS Viewer</a></li>
        
        <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
        <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
        
        
        <li><a href="https://legumeinfo.org/sequenceserver/">LIS SequenceServer BLAST</a></li>
        
        <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
        
      </ul>
    </div>
  </li>
  <li><a href="/blog">Blog</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>

</ul>
<a href="#off-screen-menu" class="uk-navbar-toggle uk-hidden@s" uk-navbar-toggle-icon uk-toggle></a>

        <ul class="uk-navbar-nav uk-visible@s">

        
          <li><a class="tm-no-underline" href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span></a></li>
        
        
        
          <li><a class="tm-no-underline" href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span></a></li>
        

        </ul>
        
        
        <ul class="uk-navbar-nav">
          <li><a class="tm-no-underline" title="Toggle alerts" uk-toggle="target: #alerts; animation: uk-animation-fade"><span uk-icon="bell"></span></a></li>
        </ul>
        
      </div>
    </nav>
    <nav class="uk-navbar tm-lower-navbar">
      <div class="uk-navbar-left">

        <ul class="uk-navbar-nav uk-hidden@l uk-visible@s">
    <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        
        <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
        
        
        
        
        
        <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <li class="uk-nav-divider"></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <li><a href="/taxa/aeschynomene"> Aeschynomene (Jointvetch)</a></li>
        
        
        
        
        
        <li><a href="/taxa/apios"> Apios (Potato bean)</a></li>
        
        
        
        
        
        <li><a href="/taxa/bauhinia"> Bauhinia (Orchid tree)</a></li>
        
        
        
        
        
        <li><a href="/taxa/cercis"> Cercis (Redbud tree)</a></li>
        
        
        
        
        
        <li><a href="/taxa/chamaecrista"> Chamaecrista (Partridge pea)</a></li>
        
        
        
        
        
        <li><a href="/taxa/faidherbia"> Faidherbia (Apple-ring acacia)</a></li>
        
        
        
        
        
        <li><a href="/taxa/zenia"> Zenia (Southeast Asian tree)</a></li>
        
        
        
      </ul>
    </div>
  </li>
  <li>
    <a href="/collections">Collections</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        
        <li> <a href="/collections/arachis"> Arachis (Peanut)</a></li>
        
        
        
        
        
        <li> <a href="/collections/cajanus"> Cajanus (Pigeonpea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/cicer"> Cicer (Chickpea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/glycine"> Glycine (Soybean)</a></li>
        
        
        
        
        
        <li> <a href="/collections/lotus"> Lotus (Bird's-foot trefoil)</a></li>
        
        
        
        
        
        <li> <a href="/collections/lupinus"> Lupinus (Lupin grain legumes)</a></li>
        
        
        
        
        
        <li> <a href="/collections/medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
        
        
        
        
        
        <li> <a href="/collections/phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
        
        
        
        
        
        <li> <a href="/collections/pisum"> Pisum (Pea)</a></li>
        
        
        
        
        
        <li> <a href="/collections/trifolium"> Trifolium (Clover)</a></li>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
      </ul>
    </div>
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
        
        
        <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
        <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
        <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
        <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
        <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
        <li><a href="https://cmap.legumeinfo.org/">CMap</a></li>
        
        <li><a href="https://legumeinfo.org/germplasm/map">LIS Germplasm GIS Viewer</a></li>
        
        <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
        <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
        
        
        <li><a href="https://legumeinfo.org/sequenceserver/">LIS SequenceServer BLAST</a></li>
        
        <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
        
      </ul>
    </div>
  </li>
  <li><a href="/blog">Blog</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>

</ul>

      </div>
    </nav>
  </div>
</div>

<div id="off-screen-menu" uk-offcanvas="overlay: true">
  <div class="uk-offcanvas-bar uk-dark  uk-background-secondary">

    <ul class="uk-nav-default uk-nav-parent-icon" uk-nav>
  <li><a href="/">Home</a></li>
  <li><a href="/taxa">Taxa</a></li>
  <li><a href="/collections">Collections</a></li>
  <li><a href="/tools">Tools</a></li>
  <li><a href="/blog">Blog</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
  <li class="uk-nav-divider"></li>
  <li><a href="/news">News</a></li>
  <li><a href="/announcements">Announcements</a></li>
</ul>


    
    <ul class="uk-nav uk-nav-default">
      <li class="uk-nav-divider"></li>
      
      <li><a href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span> Twitter</a></li>
      
      
      
      <li><a href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span> GitHub</a></li>
      
    </ul>
    

  </div>
</div>



  <div id="left-menu" class="tm-sidebar-left uk-visible@m">

  
  
  
    <h3>Data Collections</h3>

<ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
  
  
  
  <li><a href="/collections/arachis">Arachis</a></li>
  
  
  
  
  
  <li><a href="/collections/cajanus">Cajanus</a></li>
  
  
  
  
  
  <li><a href="/collections/cicer">Cicer</a></li>
  
  
  
  
  
  <li><a href="/collections/glycine">Glycine</a></li>
  
  
  
  
  
  <li><a href="/collections/lotus">Lotus</a></li>
  
  
  
  
  
  <li><a href="/collections/lupinus">Lupinus</a></li>
  
  
  
  
  
  <li><a href="/collections/medicago">Medicago</a></li>
  
  
  
  
  
  <li><a href="/collections/phaseolus">Phaseolus</a></li>
  
  
  
  
  
  <li><a href="/collections/pisum">Pisum</a></li>
  
  
  
  
  
  <li><a href="/collections/trifolium">Trifolium</a></li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
</ul>

  

  <!-- dynamically set the top style property of the left menu when the height of the navbar changes -->
  <!-- NOTE: the script tag is nested so it doesn't break a CSS selector -->
  <script type="text/javascript">

    const navbar = document.getElementById("navbar");
    const left = document.getElementById("left-menu");

    const leftTopObserver = new ResizeObserver(entries => {
      for (let entry of entries) {
        // use the newer contentBoxSize property if the browser supports it
        if (entry.contentBoxSize) {
          left.style.top = entry.contentBoxSize[0].blockSize + "px";
        // fallback to the older contentRect property
        } else {
          left.style.top = entry.contentRect.height + "px";
        }
      }
    });

    leftTopObserver.observe(navbar);

  </script>
</div>


<!-- NOTE: putting content between the left menu and tm-main blocks will break CSS selectors -->
<div class="tm-main uk-section uk-section-default uk-flex">
  <div class="uk-container uk-width-1-1">
    


<h1><i>Phaseolus</i> data collections</h1>



<h3><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/lunatus/"><i>Phaseolus lunatus</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/lunatus/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/lunatus/genomes/G27455.gnm1.7NXX/">G27455.gnm1.7NXX</a></dt>
  <dd>Phaseolus lunatus accession G27455 genome assembly V1</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/lunatus/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/lunatus/annotations/G27455.gnm1.ann1.JD7C/">G27455.gnm1.ann1.JD7C</a></dt>
  <dd>Phaseolus lunatus accession G27455 annotation files from genome assembly V1</dd>
  
</dl>












<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/"><i>Phaseolus vulgaris</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/5-593.gnm1.1P7P/">5-593.gnm1.1P7P</a></dt>
  <dd>Phaseolus vulgaris accession 5-593 (Middle American race), genome assembly v1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/G19833.gnm1.zBnF/">G19833.gnm1.zBnF</a></dt>
  <dd>Phaseolus vulgaris accession G19833 genome assembly v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/G19833.gnm2.fC0g/">G19833.gnm2.fC0g</a></dt>
  <dd>Phaseolus vulgaris accession G19833 genome assembly v2.1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/LaborOvalle.gnm1.VZZS/">LaborOvalle.gnm1.VZZS</a></dt>
  <dd>Phaseolus vulgaris accession Labor Ovalle (race Guatemala) genome assembly v1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genomes/UI111.gnm1.3VTM/">UI111.gnm1.3VTM</a></dt>
  <dd>Phaseolus vulgaris accession UI111 genome assembly v1</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/5-593.gnm1.ann1.3FBJ/">5-593.gnm1.ann1.3FBJ</a></dt>
  <dd>Phaseolus vulgaris accession 5-593 (Middle American race), annotation 1, genome assembly v1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/G19833.gnm1.ann1.pScz/">G19833.gnm1.ann1.pScz</a></dt>
  <dd>Phaseolus vulgaris accession G19833 annotation files</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/G19833.gnm2.ann1.PB8d/">G19833.gnm2.ann1.PB8d</a></dt>
  <dd>Phaseolus vulgaris accession G19833 annotation files from genome assembly 2.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/LaborOvalle.gnm1.ann1.L1DY/">LaborOvalle.gnm1.ann1.L1DY</a></dt>
  <dd>Phaseolus vulgaris accession Labor Ovalle (race Guatemala), annotation 1, genome assembly v1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/annotations/UI111.gnm1.ann1.8L4N/">UI111.gnm1.ann1.8L4N</a></dt>
  <dd>Phaseolus vulgaris accession UI111 annotation files from genome assembly 1.0</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/expression/">Expression</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/expression/G19833.gnm1.ann1.expr.4ZDQ/">G19833.gnm1.ann1.expr.4ZDQ</a></dt>
  <dd>LIS expression dataset phavu1 (An RNA-Seq based gene expression atlas of the common bean cv. Negro jamapa).</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/">Genetic Data</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/BAT93_x_JALOEEP558.gen.Blair_Cort%C3%A9s_2018/">BAT93_x_JALOEEP558.gen.Blair_Cort%C3%A9s_2018</a></dt>
  <dd>This BAT93 x JALOEEP558 map was constructed using Mapdisto v. 2.0 (Lorieux, 2012), assuming an RI model and using the create groups command both with and without anchor markers.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/BAT93_x_JALOEEP558.gen.Caldas_Blair_2009/">BAT93_x_JALOEEP558.gen.Caldas_Blair_2009</a></dt>
  <dd>Three RIL populations were used to find QTLs for seed color.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/BAT93_x_JALOEEP558.gen.Freyre_Skroch_1998/">BAT93_x_JALOEEP558.gen.Freyre_Skroch_1998</a></dt>
  <dd>A 75-member F8 mapping population derived from the cross of BAT93 with JALO EEP558 was used to produce a core linkage map for common bean.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/BAT93_x_JALOEEP558.gen.McConnell_Mamidi_2010/">BAT93_x_JALOEEP558.gen.McConnell_Mamidi_2010</a></dt>
  <dd>A Phaseolus vulgaris consensus genetic map from a recombinant inbred population.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/Cerinza_x_G24404.gen.Blair_Iriarte_2006/">Cerinza_x_G24404.gen.Blair_Iriarte_2006</a></dt>
  <dd>Agronomic performance traits were studied in a population of BC2F(3:5) introgression lines created from the cross of a Colombian large red-seeded commercial cultivar, ICA Cerinza, and a wild common bean accession, G24404.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/DOR364_x_BAT477.gen.Blair_Galeano_2012/">DOR364_x_BAT477.gen.Blair_Galeano_2012</a></dt>
  <dd>A mapping population of 113 RILs in the F5:7 generation was generated using a DOR364 by BAT477 cross.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/G2333_x_G19839.gen.Checa_Blair_2008/">G2333_x_G19839.gen.Checa_Blair_2008</a></dt>
  <dd>The G2333_x_G19839_a genetic map contains all 11 common bean linkage groups.  It has a total length of 1175 cM and 149 mapped total markers.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/ICABunsi_x_SXB405.gen.Parker_BernyMierYTeran_2020/">ICABunsi_x_SXB405.gen.Parker_BernyMierYTeran_2020</a></dt>
  <dd>Pod Dehiscence QTLs were identified using three phenotyping methods and an ICA Bunsi (PD-susceptible) x SXB 405 (PD-resistant) recombinant inbred mapping population.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gen.Blair_Cort%C3%A9s_2018/">mixed.gen.Blair_Cort%C3%A9s_2018</a></dt>
  <dd>Genetic map of markers to Phaseolus vulgaris accession G19833 genome assembly v1.0</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gen.Galeano_Fernandez_2011/">mixed.gen.Galeano_Fernandez_2011</a></dt>
  <dd>This common bean consensus map was created using the DB (DOR364_x_BAT477, Mesoamerican intra-gene pool cross), DG (DOR364_x_G19833, inter-gene pool cross), and BJ (BAT93_x_JALOEEP558, inter-gene pool cross) mapping populations.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.DiVittori_Bitocchi_2021/">mixed.gwas.DiVittori_Bitocchi_2021</a></dt>
  <dd>GWAS of pod dehiscence in Common bean, relative to the G19833 assembly 2.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Jain_Poromarto_2019/">mixed.gwas.Jain_Poromarto_2019</a></dt>
  <dd>GWAS of Soybean cyst nematode (Heterodera glycines) resistance in common bean, relative to G19833 assembly 1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Myers_Wallace_2019/">mixed.gwas.Myers_Wallace_2019</a></dt>
  <dd>GWAS of total phenolic content of pods, pod color, and flower color in Snap beans, relative to the G19833 assembly 2.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Parker_BernyMierYTeran_2020/">mixed.gwas.Parker_BernyMierYTeran_2020</a></dt>
  <dd>GWAS study of pod dehiscence in Phaseolus vulgaris dry beans, using the Andean and Mesoamerican Diversity Panels.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Raggi_Caproni_2019/">mixed.gwas.Raggi_Caproni_2019</a></dt>
  <dd>GWAS study of flowering time control in a panel of homozygous 192 common bean accessions derived from landraces.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Shi_Gepts_2021/">mixed.gwas.Shi_Gepts_2021</a></dt>
  <dd>GWAS of Soybean Cyst Nematode resistance in Common bean, relative to the the G19833 assembly 1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/mixed.gwas.Zitnick-Anderson_Oladzad_2020/">mixed.gwas.Zitnick-Anderson_Oladzad_2020</a></dt>
  <dd>GWAS of Fusarium root rot resistance in Common Bean, relative to the G19833 assembly 2.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/Stampede_x_RedHawk.gen.Song_Jia_2015/">Stampede_x_RedHawk.gen.Song_Jia_2015</a></dt>
  <dd>A mapping population of 267 F2 plants from the cross of common bean varieties Stampede and Red Hawk developed by Dr. Phil McClean at North Dakota State University.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/genetic/Xana_x_Cornell49242.gen.Gait%C3%A1n-Sol%C3%ADs_Duque_2002/">Xana_x_Cornell49242.gen.Gait%C3%A1n-Sol%C3%ADs_Duque_2002</a></dt>
  <dd>A map of 175 AFLP, 27 microsatellite, 30 SCAR, 33 ISSR, and 12 RAPD markers, as well as 13 loci coding for seed proteins, and four traditional genes (Fin/fin for growth habit, Asp/asp for seed coat shininess, P/p for seed color, and I/i for resistance to bean common mosaic virus).</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/">Markers</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.BARCBean6K/">G19833.gnm1.mrk.BARCBean6K</a></dt>
  <dd>A core set of 6000 SNPs based on the genotyping of 365 dry bean and 134 snap bean accessions with the BARCBean6K_1 and BARCBean6K_2 BeadChips.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Benchimol_deCampos_2007/">G19833.gnm1.mrk.Benchimol_deCampos_2007</a></dt>
  <dd>A common bean genomic library was constructed using the ‘IAC-UNA’ variety enriched for (CT) and (GT) for microsatellite motifs. From 1,209 sequenced clones, 714 showed microsatellites distributed over 471 simple and 243 compound motifs.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Blair_Pedraza_2003/">G19833.gnm1.mrk.Blair_Pedraza_2003</a></dt>
  <dd>A total of 150 microsatellite markers developed for common bean (Phaseolus vulgaris L.) were tested for parental polymorphism and used to determine the positions of 100 genetic loci on an integrated genetic map of the species.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Cardoso_Oblessuc_2008/">G19833.gnm1.mrk.Cardoso_Oblessuc_2008</a></dt>
  <dd>Ninety-nine new microsatelitte loci were developed from a microsatellite-enriched library for (CT)8  and (GT)8  motifs, from CAL-143 line.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Gaitan-Solis_Duque_2002/">G19833.gnm1.mrk.Gaitan-Solis_Duque_2002</a></dt>
  <dd>18 polymorphic microsatellite markers from Bactris gasipaes.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Galeano_Cortes_2012/">G19833.gnm1.mrk.Galeano_Cortes_2012</a></dt>
  <dd>In this study, a total of 313 new gene-based markers were developed at target genes.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Galeano_Fernandez_2011/">G19833.gnm1.mrk.Galeano_Fernandez_2011</a></dt>
  <dd>The DOR364 x BAT477 linkage map consisted of 291 markers with a total map length of 1,788 cM.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Grisi_Blair_2007/">G19833.gnm1.mrk.Grisi_Blair_2007</a></dt>
  <dd>61 polymorphic microsatellite markers for beans and the construction of a genetic map using the BAT93 x Jalo EEP558 (BJ) population.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Hanai_deCampos_2007/">G19833.gnm1.mrk.Hanai_deCampos_2007</a></dt>
  <dd>Three cDNA libraries generated by the Bean EST project were analyzed for the presence of SSRs. A total of 219 expressed sequence tags (ESTs) were found to carry 240 SSRs (named EST-SSR), whereas 714 genomic sequences contained 471 SSRs (named genomic-SSR).</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Hougaard_Madsen_2008/">G19833.gnm1.mrk.Hougaard_Madsen_2008</a></dt>
  <dd>One hundred and four new loci representing single-copy genes were added to the existing bean map.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Kim_Kaur_2006/">G19833.gnm1.mrk.Kim_Kaur_2006</a></dt>
  <dd>Genetic markers from Kim, Kaur, et al. (2006) mapped to G19833.gnm1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.McClean_Mamidi_2010/">G19833.gnm1.mrk.McClean_Mamidi_2010</a></dt>
  <dd>Around 15,000 common bean sequences (primarily EST contigs and EST singletons) were electronically positioned onto the common bean map using the shared syntentic blocks as references points.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Murray_Larsen_2002/">G19833.gnm1.mrk.Murray_Larsen_2002</a></dt>
  <dd>A set of 79 previously mapped bean (Phaseolus vulgaris) genomic (Bng) clones were partially sequenced. Sequence tagged site (STSs) were produced for 16 of the clones, 2 of which contain simple sequence repeats (SSRs).</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Parker_BernyMierYTeran_2020/">G19833.gnm1.mrk.Parker_BernyMierYTeran_2020</a></dt>
  <dd>Genetic markers for the GWAS dataset provided with Parker, Berny Mier y Teran (2020).</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.PvCookUCDavis2009/">G19833.gnm1.mrk.PvCookUCDavis2009</a></dt>
  <dd>A 768-marker array of single nucleotide polymorphisms (SNP) based on Trans-legume Orthologous Group (TOG) genes.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Raggi_Caproni_2019/">G19833.gnm1.mrk.Raggi_Caproni_2019</a></dt>
  <dd>Plant material was genotyped using double digest Restriction-site Associated DNA sequencing, producing about 50,000 SNPs.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm1.mrk.Yu_Park_2000/">G19833.gnm1.mrk.Yu_Park_2000</a></dt>
  <dd>15 SSR markers assigned to the Phaseolus vulgaris molecular linkage map.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm2.mrk.DiVittori_Bitocchi_2021/">G19833.gnm2.mrk.DiVittori_Bitocchi_2021</a></dt>
  <dd>We developed a BC4/F4 introgression line population and narrowed the major locus for pod shattering down to a 22.5 kb region.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/markers/G19833.gnm2.mrk.Zitnick-Anderson_Oladzad_2020/">G19833.gnm2.mrk.Zitnick-Anderson_Oladzad_2020</a></dt>
  <dd>GWA mapping using around 200,000 SNPs was performed to study Fusarium rot resistance.</dd>
  
</dl>





<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/diversity/">Diversity</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/vulgaris/diversity/G19833.gnm1.div.Moghaddam_Mamidi_2016/">G19833.gnm1.div.Moghaddam_Mamidi_2016</a></dt>
  <dd>Genotyping (variant) data for two diverse collections of Phaseolus vulgaris accessions: the Mesoamerican Diversity Panel (MDP) and the Andean Diversity Panel (ADP).</dd>
  
</dl>


<hr/>



<h3><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/"><i>Phaseolus acutifolius</i></a></h3>


<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/genomes/">Genomes</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/genomes/Frijol_Bayo.gnm1.QH8L/">Frijol_Bayo.gnm1.QH8L</a></dt>
  <dd>Phaseolus acutifolius accession Frijol Bayo, genome assembly v1.</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/genomes/W6_15578.gnm2.L2V4/">W6_15578.gnm2.L2V4</a></dt>
  <dd>Phaseolus acutifolius accession W6_15578, genome assembly v2.</dd>
  
</dl>



<h4><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/annotations/">Annotations</a></h4>
<dl>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/annotations/Frijol_Bayo.gnm1.ann1.ML22/">Frijol_Bayo.gnm1.ann1.ML22</a></dt>
  <dd>Genome annotation files for Phaseolus acutifolius accession Frijol Bayo; JGI annotation version 1 on assembly 1</dd>
  
  <dt><a target="_blank" href="https://data.legumeinfo.org/Phaseolus/acutifolius/annotations/W6_15578.gnm2.ann1.LVZ1/">W6_15578.gnm2.ann1.LVZ1</a></dt>
  <dd>Phaseolus acutifolius accession W6_15578, annotations on genome assembly v2.</dd>
  
</dl>












<hr/>




  </div>

  

</div>

</body>
</html>


</body>
</html>
