---
layout: default
title: Genomic Resources
sitemap: sequence map
tools_menu: true
---

{% capture jbrowse_base_url %}/assets/js/jbrowse{% endcapture %}
{% assign genus = "Glycine" %}

{% for about_file in site.data.datastore-metadata[genus].GENUS.about_this_collection %}
  {% if about_file[0] contains "description_" %}
    {% assign genus_resources = about_file[1] %}
    {% break %}
  {% endif %}
{% endfor %}

<h1>
  {{ genus_resources.genus }}
  ({{ genus_resources.commonName }})</h2>
<p>{{ genus_resources.description }}</p>
<p>NCBI taxonomy ID:
  <b>{{ genus_resources.taxid }}</b>
</p>

<h2>
  Overview - soybean genome and annotation statistics and nomenclature
</h2>
<p>
  Since the release of the first full soybean genome assembly in 2010, assemblies have been generated for more than 50 accessions, including multiple assemblies for the first reference, Williams 82 (Wm82).
</p>

<p>
  There are several nomenclature patterns for the assemblies and annotations. The pattern used by the DOE-JGI and SoyBase has generally taken the form Wm82.a4.v1, with the middle field ("a4") indicating assembly version and the last field (v1) indicating the annotation version. Within the SoyBase and LegumeInfo Data Store, the pattern takes the form Wm82.gnm4.ann1 -- again, with the middle field ("gnm4") indicating assembly version and the last field (ann1) indicating the annotation version.
</p>

<p>
  Access the genome and annotation data for download via the <b><a href="/collections">DATA COLLECTIONS</a> tab</b>.
</p>

<p>
  Access the genome and annotation via JBrowse the <b><a href="/resources">GENOMICS</a> tab</b>.
</p>

<p>
  See additional <b>details about the main reference assemblies at the <a href="/resources/genome_info">Genome Assembly page</a></b>.
</p>
  To examine <b>statistics about all genome assemblies and annotations held at SoyBase</b>, use these two links:
</p>
<ul>
  <li><a href="/resources/genome_stats/">Genome assembly statistics</a></li>
  <li><a href="/resources/annotation_stats/">Genome annotation statistics</a></li>
</ul>



{% if genus_resources contains "resources" %}
  <h2>Tools and resources for the genus as a whole</h2>
  <dl>
    {% for resource in genus_resources.resources %}
      <dt>
        <a href="{{ resource.URL }}">{{ resource.name }}</a>
      </dt>
      <dd>{{ resource.description }}</dd>
    {% endfor %}
  </dl>
{% endif %}

<h2>Tools and resources for particular species</h2> 

<div class="uk-flex uk-flex-center uk-flex-right@s">
<button class="uk-button uk-margin-small-right uk-button-primary" onclick="expandAllAccordion()">expand all</button>
<button class="uk-button uk-button-primary"  onclick="collapseAllAccordion()">collapse all</button>
</div>

{% for species in genus_resources.species %}
  {% for about_file in site.data.datastore-metadata[genus][species].about_this_collection %}
    {% if about_file[0] contains "description_" %}
      {% assign species_resources = about_file[1] %}
      {% break %}
    {% endif %}
  {% endfor %}

  <hr>

  <ul uk-accordion animation="false">
    <li class="ukaccordion">
  <h3 class="uk-accordion-title">
    <i>{{ species_resources.genus }} {{ species_resources.species }}</i>: {{ genus_resources.commonName }}
  </h3>
  <div class="uk-accordion-content">
  {% capture species_photo_path %}assets/img/species_images/{{ species_resources.abbrev }}.jpg{% endcapture %}
  {% assign species_photo = site.static_files | where: "path", species_photo_path | first %}
  {% if species_photo %}
    <div style="float:right;margin:10px;border:1px solid darkgreen;">
      <img src="/assets/img/species_images/{{ species_resources.abbrev }}.jpg" alt="Image: {{ species_resources.abbrev }}" /><br/>
    </div>
  {% endif %}
  <p>{{ species_resources.description }}</p>
  <p>NCBI taxonomy ID:
    <b>{{ species_resources.taxid }}</b>
  </p>

  {% if species_resources contains "resources" %}
    <h4>
      <b>
        <i>{{ species_resources.genus }} {{ species_resources.species }}</i>
        resources
      </b>
    </h4>
    <dl>
      {% for resource in species_resources.resources %}
        <dt>
          <a href="{{ resource.URL }}">{{ resource.name }}</a>
        </dt>
        <dd>{{ resource.description }}</dd>
      {% endfor %}
    </dl>
  {% endif %}

  <h4>
    <b>
      <i>{{ species_resources.genus }} {{ species_resources.species }}</i>
      accessions
    </b>
  </h4>
  {% for datatype_dir in site.data.datastore-metadata[genus][species] %}
    {% assign datatype = datatype_dir[0] %}
    {% if datatype == "about_this_collection" %}
      {% for about_file in datatype_dir[1] %}
        {% if about_file[0] contains "description_" %}
          {% assign about = about_file[1] %}
        {% endif %}
      {% endfor %}
    {% endif %}
  {% endfor %}
  {% for datatype_dir in site.data.datastore-metadata[genus][species] %}
    {% assign datatype = datatype_dir[0] %}
    {% if datatype == "annotations" %}
      {% assign existing_headers = '' | split: ',' %}
      {% for strain in about.strains %}
        
        {% unless existing_headers contains strain.accession_group %}
          <h4 class="uk-margin-small-bottom">
            <b>{{ strain.accession_group }}</b>
          </h4>
          {% if strain.accession_group contains "et al" %} 
            {% assign accession_group_type = "citation" %}

            {% comment %} 
              The following code at this level is just to assign a doi string for a citation-type accession_group_type 
            {% endcomment %}
            {% for collection_dir in datatype_dir[1] %}
              {% assign collection_name = collection_dir[0] | split: 'gnm' %}
              {% if collection_name[0] == strain.identifier %}
                {% assign collection = collection_dir[0] %}
                {% for metadata_file in collection_dir[1] %}
                  {% if metadata_file[0] contains "README" %}
                    {% assign readme = metadata_file[1] %}
                    {% assign assembly = readme.identifier | split: '.' | slice: 0, 2 | join: '.' %}
                    {% assign annotation = readme.identifier | split: '.' | slice: 0, 3 | join: '.' %}
                    {% for datatype_dir in site.data.datastore-metadata[genus][species] %}
                      {% assign datatype = datatype_dir[0] %}
                      {% if datatype == "genomes" %}
                        {% for collection_dir in datatype_dir[1] %}
                          {% assign collection = collection_dir[0] %}
                          {% for metadata_file in collection_dir[1] %}
                            {% if metadata_file[0] contains "README" %}
                              {% assign readme_genome = metadata_file[1] %}
                              {% assign assembly_genome = readme_genome.identifier | split: '.' | slice: 0, 2 | join: '.' %}
                              {% if assembly_genome == assembly %}
                                {% assign chromosome_prefix = readme_genome.chromosome_prefix %}
                                {% assign synopsis = readme_genome.synopsis %}
                                {% if readme_genome contains "publication_doi" %}
                                  {% assign doi = readme_genome.publication_doi %}
                                {% else %}
                                  {% assign doi = "DOI" %}
                                {% endif %}
                                {% break %}
                              {% endif %}
                            {% endif %}
                          {% endfor %}
                        {% endfor %}
                      {% endif %}
                    {% endfor %}
                    <dl>
                    <dd>
                      Citation (DOI) for this accession group: 
                      {% assign doiCheck = doi | slice: 0, 2 %}
                      <i>
                        {% if doiCheck == "10" %}
                          <a href="https://doi.org/{{ doi }}">doi.org/{{ doi }}</a>
                        {% else %}
                          {{ doi }}
                        {% endif %}
                      </i>
                    </dd>
                  </dl>
                  {% endif %}
                {% endfor %}
              {% endif %}
            {% endfor %}
            {% comment %} End code for assigning a doi string {% endcomment %}

          {% else %}
            {% assign accession_group_type = "reference" %}
          {% endif %}
          {% assign existing_headers = existing_headers | push: strain.accession_group %}
        {% endunless %}
        {% for collection_dir in datatype_dir[1] reversed %}
          {% assign collection_name = collection_dir[0] | split: 'gnm' %}
          {% if collection_name[0] == strain.identifier %}
            {% assign collection = collection_dir[0] %}
            {% for metadata_file in collection_dir[1] %}
              {% if metadata_file[0] contains "README" %}
                {% assign readme = metadata_file[1] %}
                {% assign assembly = readme.identifier | split: '.' | slice: 0, 2 | join: '.' %}
                {% assign annotation = readme.identifier | split: '.' | slice: 0, 3 | join: '.' %}
                {% for datatype_dir in site.data.datastore-metadata[genus][species] %}
                  {% assign datatype = datatype_dir[0] %}
                  {% if datatype == "genomes" %}
                    {% for collection_dir in datatype_dir[1] %}
                      {% assign collection = collection_dir[0] %}
                      {% for metadata_file in collection_dir[1] %}
                        {% if metadata_file[0] contains "README" %}
                          {% assign readme_genome = metadata_file[1] %}
                          {% assign assembly_genome = readme_genome.identifier | split: '.' | slice: 0, 2 | join: '.' %}
                          {% if assembly_genome == assembly %}
                            {% assign chromosome_prefix = readme_genome.chromosome_prefix %}
                            {% assign synopsis = readme_genome.synopsis %}
                            {% if readme_genome contains "publication_doi" %}
                              {% assign doi = readme_genome.publication_doi %}
                            {% else %}
                              {% assign doi = "" %}
                            {% endif %}
                          {% endif %}
                        {% endif %}
                      {% endfor %}
                    {% endfor %}
                  {% endif %}
                {% endfor %}
                <span>
                  <b>{{ assembly }}</b>

                {% comment %} 
                  Print DOI only when accession_group is of type reference, not for type citation (handled previously) 
                {% endcomment %} 
                {% if accession_group_type == "reference" %}
                 <dl>
                  <dd>{{ synopsis }}
                    {% assign doiCheck = doi | slice: 0, 2 %}
                    <i>
                      {% if doiCheck == "10" %}
                        <a href="https://doi.org/{{ doi }}">doi.org/{{ doi }}</a>
                      {% else %}
                        {{ doi }}
                      {% endif %}
                    </i>
                  </dd>
                 </dl>
                {% else %}
                  <dl><dd>{{ synopsis }}</dd></dl>
                {% endif %}
                </span>
                <div class="uk-margin-left">
               <dl>
                <dd>
                  <a target="_blank" href="{{ jbrowse_base_url }}/?assembly={{ assembly }}&tracks={{ annotation }}&loc={{ readme.scientific_name_abbrev }}.{{ assembly }}.{{ chromosome_prefix }}01:1-1000000">Genome Browser</a>
                </dd>
               </dl>
                </div>
              {% endif %}
            {% endfor %}
          {% endif %}
        {% endfor %}
      {% endfor %}
    {% endif %}
  {% endfor %}
    </div>
  </li>
</ul>
{% endfor %}


<script src="../assets/js/utils.js"></script>