<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Glycine &#8211; LIS: Legume Information System</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="A collaborative, community resource to facilitate crop improvement by integrating genetic, genomic, and trait data across legume species.">
  <meta name="robots" content="all">
  <meta name="author" content="">
  
  
  <link rel="canonical" href="/taxa/glycine/">

  
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-49655154-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-49655154-1');
</script>



  <!-- Custom CSS -->
  <link rel="stylesheet" href="/assets/uikit/dist/css/uikit.min.css" type="text/css" />
  <link rel="stylesheet" href="/assets/css/uikit-theme.css" type="text/css" />

  <!-- place globally-included stylesheet content here -->
<style>
  .species-image {
      float: right;
      margin: 10px;
  }
  .species-image img {
      border: 1px solid darkgreen;
  }
  .species-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .blog-image {
      float: right;
      margin: 10px;
  }
  .blog-image img {
      border: 1px solid gray;
  }
  .blog-image .attribution {
      text-align: center; 
      font-style: italic;
      font-size: 80%;
  }
  .temp {
      color: darkred;
      font-family: serif;
      font-weight: bold;
  }
</style>



  <!-- Custom JS -->
  <script src="/assets/uikit/dist/js/uikit.min.js"></script>
  <script src="/assets/uikit/dist/js/uikit-icons.min.js"></script>

  <!-- place globally-included scripts here -->


  <!-- opt-in code for integrating GraphQL with LIS Web Components -->
  <script type="importmap">
    {
      "imports": {
        "lis-graphql": "/assets/js/graphql/index.js"
      }
    }
  </script>

  

  <!-- Icons -->
  <link rel="icon" type="image/png" sizes="16x16" href="/assets/icons/favicon-16x16.png">
      <link rel="icon" type="image/png" sizes="192x192" href="/assets/icons/favicon-192x192.png">
      <link rel="icon" href="/assets/icons/favicon.ico">
      </head>


<body>




<div id="navbar" class="uk-navbar-container tm-navbar-container uk-sticky uk-sticky-fixed uk-light" uk-sticky>

  <div id="alerts">
    
    
      

    
  </div>

  <script type="text/javascript">

    // define the cookie key
    const alertsKey = "lis-navbar-alerts";

    // get the alerts element
    const alerts = document.getElementById("alerts");

    // define a callback that saves the toggle state
    function toggleCallback(event) {
      document.cookie = `${alertsKey}=${event.type};SameSite=Lax;path=/`;
    }

    // use the callback every time the element is toggled
    alerts.addEventListener("show", toggleCallback);
    alerts.addEventListener("hide", toggleCallback);

    // load the element's state
    const alertsCookie = document.cookie
      .split('; ')
      .find(row => row.startsWith(`${alertsKey}=`));

    // hide the alerts if they're supposed to be hidden
    if (alertsCookie && alertsCookie.split('=')[1].toLowerCase() === "hide") {
      UIkit.toggle(alerts).toggle();
    }

  </script>

  <div class="uk-container uk-container-expand">
    <nav class="uk-navbar">
      <div class="uk-navbar-left uk-text-truncate">
        <a class="uk-navbar-item uk-logo uk-text-truncate" href="/">
          
          <img class="uk-margin-small-right" src="/assets/img/logo.png" width="47" height="47" />
          
          <div class="uk-text-truncate">
            LIS: Legume Information System
            
            <div class="uk-navbar-subtitle uk-text-truncate">Information about legume traits for crop improvement</div>
            
          </div>
        </a>
      </div>
      <div class="uk-navbar-right uk-flex-none">

        <ul class="uk-navbar-nav uk-visible@l">
  <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          
            <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
      </ul>
    </div>
    
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
      </ul>
    </div>
    
  </li>
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>

<a href="#off-screen-menu" class="uk-navbar-toggle uk-hidden@s" uk-navbar-toggle-icon uk-toggle></a>


        <ul class="uk-navbar-nav uk-visible@s">

        
          <li><a class="tm-no-underline" href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span></a></li>
        
        
        
          <li><a class="tm-no-underline" href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span></a></li>
        

        </ul>
        
        
      </div>
    </nav>
    <nav class="uk-navbar tm-lower-navbar">
      <div class="uk-navbar-left">

        <ul class="uk-navbar-nav uk-hidden@l uk-visible@s">
  <li><a href="/">Home</a></li>
  <li>
    <a href="/taxa">Taxa</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          
            <li> <a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li> <a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
      </ul>
    </div>
    
  </li>
  <li>
    <a href="/tools">Tools</a>
    
    
    <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="delay-hide: 0;">
      <ul class="uk-nav uk-navbar-dropdown-nav">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
      </ul>
    </div>
    
  </li>
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>


      </div>
    </nav>
  </div>
</div>

<div id="off-screen-menu" uk-offcanvas="overlay: true">
  <div class="uk-offcanvas-bar uk-dark  uk-background-secondary">

    <ul class="uk-nav-default uk-nav-parent-icon" uk-nav>
  <li><a href="/">Home</a></li>
  
  <li class="uk-parent">
    <a href="#">Taxa</a>
    
    <ul class="uk-nav-sub">
      
        
          
            <li><a href="/taxa//arachis"> Arachis (Peanut)</a></li>
          
        
      
        
          
            <li><a href="/taxa//cajanus"> Cajanus (Pigeonpea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//cicer"> Cicer (Chickpea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//glycine"> Glycine (Soybean)</a></li>
          
        
      
        
          
            <li><a href="/taxa//lotus"> Lotus (Bird's-foot trefoil)</a></li>
          
        
      
        
          
            <li><a href="/taxa//lupinus"> Lupinus (Lupin grain legumes)</a></li>
          
        
      
        
          
            <li><a href="/taxa//medicago"> Medicago (Alfalfa/Lucerne, Barrel Medic)</a></li>
          
        
      
        
          
            <li><a href="/taxa//phaseolus"> Phaseolus (Common Bean, Lima Bean, Tepary Bean, etc.)</a></li>
          
        
      
        
          
            <li><a href="/taxa//pisum"> Pisum (Pea)</a></li>
          
        
      
        
          
            <li><a href="/taxa//trifolium"> Trifolium (Clover)</a></li>
          
        
      
        
          
            <li><a href="/taxa//vigna"> Vigna (Cowpea)</a></li>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
      <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
    </ul>
  </li>
  
  
  <li class="uk-parent">
    <a href="/tools">Tools</a>
    
    <ul class="uk-nav-sub">
      
        
          <li><a href="https://data.legumeinfo.org/">LIS Datastore</a></li>
        
          <li><a href="https://mines.legumeinfo.org/">LIS InterMines</a></li>
        
          <li><a href="https://dscensor.legumeinfo.org/">DSCensor Data Overviews</a></li>
        
      
        
          <li><a href="/tools/search/gene.html">Gene Search Tool</a></li>
        
      
        
          <li><a href="https://sequenceserver.legumeinfo.org/">LIS SequenceServer BLAST</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/">Funnotate: Annotate Your Sequences</a></li>
        
      
        
          <li><a href="https://gcv.legumeinfo.org/">Genome Context Viewer</a></li>
        
          <li><a href="https://zzbrowse.legumeinfo.org/">ZZBrowse GWAS/QTL</a></li>
        
          <li><a href="https://cmap.legumeinfo.org/">CMap: Genetic Map Viewer</a></li>
        
          <li><a href="https://germplasm-map.legumeinfo.org/">LIS Germplasm GIS Viewer</a></li>
        
          <li><a href="https://conekt.legumeinfo.org/species">Conekt Comparative Expression</a></li>
        
          <li><a href="https://funnotate.legumeinfo.org/?search">Gene Family Search</a></li>
        
          <li><a href="/tools/gcvit">GCViT: SNP Comparison Tool</a></li>
        
      
    </ul>
  </li>
  
  <li><a href="/download">Download</a></li>
  <li><a href="/community">Community</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>


    
    <ul class="uk-nav uk-nav-default">
      <li class="uk-nav-divider"></li>
      
      <li><a href="https://twitter.com/legumeinfo" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span> Twitter</a></li>
      
      
      
      <li><a href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span> GitHub</a></li>
      
    </ul>
    

  </div>
</div>



  <div id="left-menu" class="tm-sidebar-left uk-visible@m">

  
  
    
<h3>Taxa</h3>
<ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
  
  
  
  <li><a href="/taxa/arachis">Arachis</a></li>
  
  
  
  
  
  <li><a href="/taxa/cajanus">Cajanus</a></li>
  
  
  
  
  
  <li><a href="/taxa/cicer">Cicer</a></li>
  
  
  
  
  
  <li><a href="/taxa/glycine">Glycine</a></li>
  
  
  
  
  
  <li><a href="/taxa/lotus">Lotus</a></li>
  
  
  
  
  
  <li><a href="/taxa/lupinus">Lupinus</a></li>
  
  
  
  
  
  <li><a href="/taxa/medicago">Medicago</a></li>
  
  
  
  
  
  <li><a href="/taxa/phaseolus">Phaseolus</a></li>
  
  
  
  
  
  <li><a href="/taxa/pisum">Pisum</a></li>
  
  
  
  
  
  <li><a href="/taxa/trifolium">Trifolium</a></li>
  
  
  
  
  
  <li><a href="/taxa/vigna">Vigna</a></li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="uk-nav-divider"></li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li><a href="/taxa/aeschynomene">Aeschynomene</a></li>
  
  
  
  
  
  <li><a href="/taxa/apios">Apios</a></li>
  
  
  
  
  
  <li><a href="/taxa/cercis">Cercis</a></li>
  
  
  
  
  
  <li><a href="/taxa/chamaecrista">Chamaecrista</a></li>
  
  
  
  
  
  <li><a href="/taxa/faidherbia">Faidherbia</a></li>
  
  
  
</ul>

  
  

  <!-- dynamically set the top style property of the left menu when the height of the navbar changes -->
  <!-- NOTE: the script tag is nested so it doesn't break a CSS selector -->
  <script type="text/javascript">

    const navbar = document.getElementById("navbar");
    const left = document.getElementById("left-menu");

    const leftTopObserver = new ResizeObserver(entries => {
      for (let entry of entries) {
        // use the newer contentBoxSize property if the browser supports it
        if (entry.contentBoxSize) {
          left.style.top = entry.contentBoxSize[0].blockSize + "px";
        // fallback to the older contentRect property
        } else {
          left.style.top = entry.contentRect.height + "px";
        }
      }
    });

    leftTopObserver.observe(navbar);

  </script>
</div>


<!-- NOTE: putting content between the tools and tm-main blocks will break CSS selectors -->
<div class="tm-main uk-section uk-section-default uk-flex">
  <div class="uk-container uk-width-1-1">
    





<h1><i>Glycine</i> (soybean)</h2>

<p>The best-known species in Glycine is the cultivated soybean, G. max, which was domesticated in Central and East Asia. The majority of the species in the genus are found only in Australia, while a few species extend from Australia to East Asia.</p>
<p>NCBI taxonomy ID: <b>3847</b></p>


<h2>Tools and resources for the genus as a whole</h2>
<dl>
  
  <dt><a href="https://mines.legumeinfo.org/glycinemine/begin.do">GlycineMine</a></dt>
  <dd>InterMine interface for accessing genetic and genomic data for several species in Glycine.</dd>
  
  <dt><a href="https://zzbrowse.legumeinfo.org/?tab=WhGen&datasets=Peanut%20GWAS&chr=Arahy.01&selected=100000&window=250000&datasets2=Soybean%20GWAS&chr2=Gm01&selected2=100000&window2=250000">ZZBrowse</a></dt>
  <dd>Association viewers (QTL, GWAS)</dd>
  
  <dt><a href="https://soybase.org/gcvit">GCViT</a></dt>
  <dd>Genotype comparison visualization tool</dd>
  
  <dt><a href="https://gcv.legumeinfo.org/gene;lis=glyma.Glyma.15G255600?q=Glyma.15G255600&sources=lis&algorithm=repeat&match=10&mismatch=-1&gap=-1&score=30&threshold=25&bmatched=20&bintermediate=10&bmask=10&linkage=average&cthreshold=20&neighbors=10&matched=4&intermediate=5&bregexp=&border=chromosome&regexp=&order=distance">Genome Context Viewer</a></dt>
  <dd>Browser for dynamically discovering and viewing genomic synteny across selected species.</dd>
  
  <dt><a href="https://soybase.org/grindata/">Grin Data Explorer</a></dt>
  <dd>Tool to facilitate searches of GRIN Descriptor Data</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMapII project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h2>Tools and resources for particular species</h2>



<hr/>

<h3 id="max"><i>Glycine max</i>: soybean</h3>






<p>Soybean (Glycine max), the predominant oil-seed legume worldwide, was likely domesticated in East Asia, ~6000-9000 years ago (Sedivy et al., 2017; https://doi.org/10.1111/nph.14418). It has many culinary and industrial uses. Some of the culinary uses include: for direct consumption of the green seed (i.e. edamame) and leaves (cooked, much like spinach); for tofu, soymilk, textured vegetable protein, soy sauce, tempeh, natto, and vegetable oil. Industrial uses include: oils, soap, cosmetics, and biodiesel. Soybean is also used as a high-protein forage, and can be prepared for fish- and animal-feed.</p>

<p>NCBI taxonomy ID: <b>3847</b></p>


<h4><i>Glycine max</i> resources</h4>
<dl>
  
  <dt><a href="https://mines.legumeinfo.org/glycinemine/begin.do">GlycineMine</a></dt>
  <dd>InterMine interface for accessing genetic and genomic data for several species in Glycine.</dd>
  
  <dt><a href="https://zzbrowse.legumeinfo.org/?tab=WhGen&datasets=Peanut%20GWAS&chr=Arahy.01&selected=100000&window=250000&datasets2=Soybean%20GWAS&chr2=Gm01&selected2=100000&window2=250000">ZZBrowse</a></dt>
  <dd>Association viewers (QTL, GWAS)</dd>
  
  <dt><a href="https://soybase.org/gcvit">GCViT</a></dt>
  <dd>Genotype comparison visualization tool </dd>
  
  <dt><a href="https://gcv.legumeinfo.org/gene;lis=glyma.Glyma.15G255600?q=Glyma.15G255600&sources=lis&algorithm=repeat&match=10&mismatch=-1&gap=-1&score=30&threshold=25&bmatched=20&bintermediate=10&bmask=10&linkage=average&cthreshold=20&neighbors=10&matched=4&intermediate=5&bregexp=&border=chromosome&regexp=&order=distance">Genome Context Viewer</a></dt>
  <dd>Browser for dynamically discovering and viewing genomic synteny across selected species.</dd>
  
  <dt><a href="https://soybase.org/grindata/">Grin Data Explorer</a></dt>
  <dd>Tool to facilitate searches of GRIN Descriptor Data</dd>
  
</dl>


<h4><i>Glycine max</i> accessions</h4>

<b>Wm82_ISU01</b> (Northern United States) <br/>
Glycine max var. Williams 82-ISU-01 is an inbred line developed by Robert Stupar (University of Minnesota) by inbreeding original W82 seed. The resulting genome is a haploid version of W82. This genome can be used as an updated version of W82. It is the most complete and highest quality soybean genome completed to date.

<br/><i>No resources are currently listed for accession <b>Wm82_ISU01</b></i>.


<b>Wm82</b> (Northern United States) <br/>
Williams 82, the soybean cultivar used to produce the reference genome sequence, was derived from backcrossing a phytophthora root rot resistance locus from the donor parent Kingwa into the recurrent parent Williams.

<br/><i>No resources are currently listed for accession <b>Wm82</b></i>.


<b>Lee</b> (Southern United States) <br/>
Cultivar Lee, which derives from a cross of Chinese lines CNS and S-100, has been widely used as a parent in many breeding projects in the Southern U.S. and in Brazil (Wysmierski and Vello, 2013; http://doi.org/10.1590/S1415-47572013005000041). The variety is notable for resistance to Phytophthora rot, Peanut Mottle Virus, and bacterial pustule (Wysmierski and Vello, 2013).

<br/><i>No resources are currently listed for accession <b>Lee</b></i>.


<b>FiskebyIII</b> (Fiskeby, Norrköpping, Sweden) <br/>
Bred by the late Dr. Sven Holmberg in Fiskeby, Norrköpping, Sweden. Highly nutritious — up to 40% protein, high in calcium, iron, and vitamins (particularly A, B1, B12, and C). Thrives in northern climates. 75-80 days.

<br/><i>No resources are currently listed for accession <b>FiskebyIII</b></i>.


<b>Zh13</b> (China) <br/>
Zhonghuang 13 is a Chinese cultivar derived from accessions Yudou 18 and Zhongzuo 90052-76 by pedigree selection for high yield and stress tolerance (Shen et al., 2018; https://doi.org/10.1007/s11427-018-9360-0).

<br/><i>No resources are currently listed for accession <b>Zh13</b></i>.


<b>Hwangkeum</b> (South Korea) <br/>
Hwangkeum is an important South Korean cultivar, released in 1979 and widely used since then as a breeding parent in Korea. Hwangkeum has a determinate growth habit and nonshattering pods and is adapted to the middle Korean peninsula (Maturity Group V).

<br/><i>No resources are currently listed for accession <b>Hwangkeum</b></i>.


<b>JD17</b> (China) <br/>
Jidou 17 is an Chinese elite cultivar, widely grown in central China (in maturity group III). This variety is recognized for high yields and tolerance for high planting densities.

<br/><i>No resources are currently listed for accession <b>JD17</b></i>.


<b>Hefeng25_IGA1002</b> (Jiamusi Branch of Heilongjiang Academy of Agricultural Sciences) <br/>
Glycine max Hefeng 25 (IGA1002) was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Hefeng25_IGA1002</b></i>.


<b>Huaxia3_IGA1007</b> (South China Agricultural University) <br/>
Glycine max Huaxia 3 (IGA1007) was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Huaxia3_IGA1007</b></i>.


<b>Jinyuan_IGA1006</b> (National Soybean Preservation Center, ICS, CAAS) <br/>
Glycine max Jinyuan (IGA1006) was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Jinyuan_IGA1006</b></i>.


<b>Wenfeng7_IGA1001</b> (National Soybean Preservation Center, ICS, CAAS) <br/>
Glycine max Wenfeng 7 (IGA1001) was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Wenfeng7_IGA1001</b></i>.


<b>Wm82_IGA1008</b> (Northeast Institute of Geography and Agroecology, CAS) <br/>
Williams 82, the soybean cultivar used to produce the reference genome sequence, was derived from backcrossing a phytophthora root rot resistance locus from the donor parent Kingwa into the recurrent parent Williams. This line, IGA1008, was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Wm82_IGA1008</b></i>.


<b>Zh13_IGA1005</b> (National Soybean Improvement Sub-Center, ICS, CAAS) <br/>
Zhonghuang 13 is a Chinese cultivar derived from accessions Yudou 18 and Zhongzuo 90052-76 by pedigree selection for high yield and stress tolerance (Shen et al., 2018; https://doi.org/10.1007/s11427-018-9360-0). This line, IGA1005, was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Zh13_IGA1005</b></i>.


<b>Zh35_IGA1004</b> (National Soybean Improvement Sub-Center, ICS, CAAS) <br/>
Glycine max Zhonghuang 35 (IGA1004) was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>Zh35_IGA1004</b></i>.


<b>58-161</b> (China, Jiang Su) <br/>
Landrace 58-161 (SoyL04 from China, Jiang Su, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>58-161</b></i>.


<b>Amsoy</b> (United States, Iowa) <br/>
Cultivar Amsoy (SoyC05 from United States, Iowa, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>Amsoy</b></i>.


<b>DongNongNo_50</b> (China, Hei Long Jiang) <br/>
Cultivar Dong Nong No.50 (SoyC12 from China, Hei Long Jiang, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>DongNongNo_50</b></i>.


<b>FengDiHuang</b> (China, Ji Lin) <br/>
Landrace Feng Di Huang (SoyL07 from China, Ji Lin, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>FengDiHuang</b></i>.


<b>HanDouNo_5</b> (China, He Bei) <br/>
Cultivar Han Dou No.5 (SoyC09 from China, He Bei, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>HanDouNo_5</b></i>.


<b>HeiHeNo_43</b> (China, Hei Long Jiang) <br/>
Cultivar Hei He No.43  (SoyC13 from China, Hei Long Jiang, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>HeiHeNo_43</b></i>.


<b>JiDouNo_17</b> (China, He Bei) <br/>
Cultivar Ji Dou No.17 (SoyC11 from China, He Bei, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>JiDouNo_17</b></i>.


<b>JinDouNo_23</b> (China, Shan Xi) <br/>
Cultivar Jin Dou No.23 (SoyC07 from China, Shan Xi, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>JinDouNo_23</b></i>.


<b>JuXuanNo_23</b> (China, Shan Dong) <br/>
Cultivar Ju Xuan No.23 (SoyC03 from China, Shan Dong, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>JuXuanNo_23</b></i>.


<b>KeShanNo_1</b> (China, Hei Long Jiang) <br/>
Cultivar Ke Shan No.1 (SoyC14 from China, Hei Long Jiang, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>KeShanNo_1</b></i>.


<b>PI_398296</b> (South Korea, Kyonggi) <br/>
Landrace PI 398296 (SoyL05 from South Korea, Kyonggi, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>PI_398296</b></i>.


<b>PI_548362</b> (United States, Illinois) <br/>
Cultivar PI 548362 (SoyC10 from United States, Illinois, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>PI_548362</b></i>.


<b>QiHuangNo_34</b> (China, Shan Dong) <br/>
Cultivar Qi Huang No.34 (SoyC08 from China, Shan Dong, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>QiHuangNo_34</b></i>.


<b>ShiShengChangYe</b> (Japan, Hokkaido) <br/>
Landrace Shi Sheng Chang Ye (SoyL09 from Japan, Hokkaido, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>ShiShengChangYe</b></i>.


<b>TieFengNo_18</b> (China, Hei Long Jiang) <br/>
Cultivar Tie Feng No.18 (SoyC02 from China, Hei Long Jiang, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>TieFengNo_18</b></i>.


<b>TieJiaSiLiHuang</b> (China, Ji Lin) <br/>
Landrace Tie Jia Si Li Huang (SoyL08 from China, Ji Lin, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>TieJiaSiLiHuang</b></i>.


<b>TongShanTianEDan</b> (China, Jiang Su) <br/>
Landrace Tong Shan Tian E Dan (SoyL03 from China, Jiang Su, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>TongShanTianEDan</b></i>.


<b>WanDouNo_28</b> (China, An Hui) <br/>
Cultivar Wan Dou No.28 (SoyC04 from China, An Hui, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>WanDouNo_28</b></i>.


<b>XuDouNo_1</b> (China, Jiang Su) <br/>
Cultivar Xu Dou No.1 (SoyC01 from China, Jiang Su, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>XuDouNo_1</b></i>.


<b>YuDouNo_22</b> (China, He Nan) <br/>
Cultivar Yu Dou No.22 (SoyC06 from China, He Nan, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>YuDouNo_22</b></i>.


<b>ZhangChunManCangJin</b> (China, Ji Lin) <br/>
Landrace Chang Chun Man Cang Jin (SoyL06 from China, Ji Lin, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>ZhangChunManCangJin</b></i>.


<b>Zhutwinning2</b> (China) <br/>
Landrace Zhutwinning2 (SoyL01 from China, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>Zhutwinning2</b></i>.


<b>ZiHuaNo_4</b> (China, Hei Long Jiang) <br/>
Landrace Zi Hua No.4 (SoyL02 from China, Hei Long Jiang, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>ZiHuaNo_4</b></i>.


<b>Wm82_NJAU</b> (Nanjing Agricultural University) <br/>
Williams 82 sequenced for T2T assembly of Wang, Zhang et al. 2023

<br/><i>No resources are currently listed for accession <b>Wm82_NJAU</b></i>.





<hr/>

<h3 id="soja"><i>Glycine soja</i>: wild soybean</h3>






<p>Glycine soja is the closest wild relative of soybean, Glycine max. Populations of G. soja exist in the wild in China, Japan, Korea, and Russia. Analysis of genetic differences between the two species suggests that the two separated approximately 200 thousand years ago. The species remain interfertile, and G. soja accessions are used in breeding projects in order to introgress traits such as tolerance to particular diseases or environmental stresses.</p>

<p>NCBI taxonomy ID: <b>3848</b></p>


<h4><i>Glycine soja</i> resources</h4>
<dl>
  
  <dt><a href="https://mines.legumeinfo.org/glycinemine/begin.do">GlycineMine</a></dt>
  <dd>InterMine interface for accessing genetic and genomic data for several species in Glycine.</dd>
  
  <dt><a href="https://zzbrowse.legumeinfo.org/?tab=WhGen&datasets=Peanut%20GWAS&chr=Arahy.01&selected=100000&window=250000&datasets2=Soybean%20GWAS&chr2=Gm01&selected2=100000&window2=250000">ZZBrowse</a></dt>
  <dd>Association viewers (QTL, GWAS)</dd>
  
  <dt><a href="https://soybase.org/gcvit">GCViT</a></dt>
  <dd>Genotype comparison visualization tool </dd>
  
  <dt><a href="https://gcv.legumeinfo.org/gene;lis=glyso.PI483463.gnm1.ann1.GlysoPI483463.08G318500">Genome Context Viewer</a></dt>
  <dd>Browser for dynamically discovering and viewing genomic synteny across selected species.</dd>
  
  <dt><a href="https://soybase.org/grindata/">Grin Data Explorer</a></dt>
  <dd>Tool to facilitate searches of GRIN Descriptor Data</dd>
  
</dl>


<h4><i>Glycine soja</i> accessions</h4>

<b>PI483463</b> (Shanxi Province, China) <br/>
Glycine soja accession PI 483463 has been identified as being unusually tolerant (Lee et al., 2009; https://doi.org/10.1093/jhered/esp027). The genome for this accession has been sequenced, partly on the basis of this salt-tolerance trait.

<br/><i>No resources are currently listed for accession <b>PI483463</b></i>.


<b>W05</b> (Henan Province, China) <br/>
Glycine soja accession W05 is a salt-tolerant wild soybean whose genome has been sequenced to serve as a reference genome assembly. The W05 accession has been used for genetic studies of several traits, including indeterminacy, seed size, pod number per plant, and seed color (Xie et al., 2019; https://doi.org/10.1038/s41467-019-09142-9).

<br/><i>No resources are currently listed for accession <b>W05</b></i>.


<b>F_IGA1003</b> (National Wild Soybean Preservation Center, ICS, CAAS) <br/>
This Glycine soja wild soybean accession was sequenced and annotated by Chu, et al. (2021).

<br/><i>No resources are currently listed for accession <b>F_IGA1003</b></i>.


<b>PI_549046</b> (China, Shaan Xi) <br/>
Wild (G. soja) accession PI 549046 (SoyW02 from China, Shaan Xi, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>PI_549046</b></i>.


<b>PI_562565</b> (South Korea, Jeolla Puk) <br/>
Wild (G. soja) accession PI 562565 (SoyW01 from South Korea, Jeolla Puk, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>PI_562565</b></i>.


<b>PI_578357</b> (Russian Federation, Federation Amur) <br/>
Wild (G. soja) accession PI 578357 (SoyW03 from Russian Federation, Federation Amur, from Liu et al. 2020

<br/><i>No resources are currently listed for accession <b>PI_578357</b></i>.





<hr/>

<h3 id="cyrtoloba"><i>Glycine cyrtoloba</i></h3>






<p>G. cyrotoloba (Tind) is a perennial plant with twining and stiff stems. G. cyrotoloba pods are curved and somewhat mottled in appearance containing 3-9 seeds that and are dark brown to black in color (Tindale, MD et al., 1984). G. cyrotoloba is a diploid (2n=40) member of the C-genome of Glycine. It is found along the coast of Queensland and Northern New South Wales (Ratnaparkhe et al 2011 ; Gonzalez-Orozco et al., 2012).</p>

<p>NCBI taxonomy ID: <b>45689</b></p>


<h4><i>Glycine cyrtoloba</i> resources</h4>
<dl>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_cyrtoloba_BES_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. cyrtoloba Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_cyrtoloba_bes_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. cyrtoloba Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
</dl>


<h4><i>Glycine cyrtoloba</i> accessions</h4>

<b>G1267</b> (Australian coasts of Queensland and Northern New South Wales) <br/>
G. cyrotoloba (Tind) is a perennial plant with twining and stiff stems. G. cyrotoloba pods are curved and somewhat mottled in appearance containing 3-9 seeds that and are dark brown to black in color (Tindale, MD et al., 1984).

<br/><i>No resources are currently listed for accession <b>G1267</b></i>.





<hr/>

<h3 id="dolichocarpa"><i>Glycine dolichocarpa</i></h3>






<p>Glycine dolichocarpa (Tateishi & Ohashi) is a twining plant with long straight dark brown pods with 5-7 seeds. Seeds are square and dark brown in color. G. dolichocarpa is an allotetraploid (2n = 4x = 80) formed by hybridizatoin between G. syndetika and G. tomentella D3 (both 2n = 40). (This species was formerly part of the Glycine tomentella species complex and was referred to as G. tomentella T2.) It has a limited Australian range in Queensland, but like several other Glycine allopolyploids, has colonized islands of the Pacific Ocean (in this case Taiwan) where no perennial diploid Glycine species have been found ( Ratnaparkhe et al 2011; Harbert et al 2014).</p>

<p>NCBI taxonomy ID: <b>82538</b></p>


<h4><i>Glycine dolichocarpa</i> resources</h4>
<dl>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_dolichocarpa_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. dolichocarpa Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_dolichocarpa_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. dolichocarpa Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h4><i>Glycine dolichocarpa</i> accessions</h4>

<b>G1134</b> (Queensland, Australia) <br/>
Glycine dolichocarpa (Tateishi & Ohashi) is a twining plant with long straight dark brown pods with 5-7 seeds. Seeds are square and dark brown in color.

<br/><i>No resources are currently listed for accession <b>G1134</b></i>.





<hr/>

<h3 id="falcata"><i>Glycine falcata</i></h3>






<p>Glycine falcata (Benth.) is unique among perennial Glycine species in that it does not form a vine but rather short, erect stems from a fibrous woody root system instead of the more common taproot. Seeds are round and smooth similar to the annual species. G. falcata is a diploid (2n = 40) and is the sole member of the F-genome. It is sister to the remainder of subgenus Glycine, and is distinctive ecologically, characteristically growing in the black soil region of Queensland and possessing both chasmogamous and below- ground cleistogamous flowers, the latter producing geocarpic fruits (Ratnaparkhe et al 2011; Gonzalez-Orozco et al., 2012).</p>

<p>NCBI taxonomy ID: <b>45690</b></p>


<h4><i>Glycine falcata</i> resources</h4>
<dl>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_falcata_BES_alignments">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. falcata Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_falcata_bes_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. falcata Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h4><i>Glycine falcata</i> accessions</h4>

<b>G1718</b> (Queensland, Australia) <br/>
Glycine falcata (Benth.) is unique among perennial Glycine species in that it does not form a vine but rather short, erect stems from a fibrous woody root system instead of the more common taproot. Seeds are round and smooth similar to the annual species.

<br/><i>No resources are currently listed for accession <b>G1718</b></i>.





<hr/>

<h3 id="stenophita"><i>Glycine stenophita</i></h3>






<p>Glycine stenophita (B.E. Pfeil & Tind.) is a scrambling or climbing perennial that is glabrous or with sparse white hairs covering the stems. Pods are 4 to 6 seeded and seeds are generally barrel shaped with some variation in shape from elliptical to square. G. stenophita is a diploid (2n = 40) member of the B-genome group. It occurs in the Australian states of Queensland and New South Wales (Ratnaparkhe et al 2011; Gonzalez-Orozco et al., 2012).</p>

<p>NCBI taxonomy ID: <b>96944</b></p>


<h4><i>Glycine stenophita</i> resources</h4>
<dl>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_stenophita_BES_alignments">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. stenophita Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_stenophita_bes_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. stenophita Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h4><i>Glycine stenophita</i> accessions</h4>

<b>G1974</b> (Australian states of Queensland and New South Wales) <br/>
Glycine stenophita (B.E. Pfeil & Tind.) is a scrambling or climbing perennial that is glabrous or with sparse white hairs covering the stems. Pods are 4 to 6 seeded and seeds are generally barrel shaped with some variation in shape from elliptical to square.

<br/><i>No resources are currently listed for accession <b>G1974</b></i>.





<hr/>

<h3 id="syndetika"><i>Glycine syndetika</i></h3>






<p>Glycine syndetika (B.E. Pfeil & Craven) is a twining perennial plant with three leathery, often persistent leaflets. Flowers are somewhat clustered towards to the top of the inflorescences and pods contain 4-9 relatively large square seeds (Pfeil. BE et al., 2006). G. syndetika is diploid (2n = 40) member of the A-genome clade. (This species was formerly part of the Glycine tomentella species complex and was referred to as G. tomentella D4. ) It is has a restricted range in the Eastern Queensland region of Australia (Ratnaparkhe et al 2011; Gonzalez-Orozco et al., 2012).</p>

<p>NCBI taxonomy ID: <b>713886</b></p>


<h4><i>Glycine syndetika</i> resources</h4>
<dl>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_syndetika_BES_alignments">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. syndetika Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_syndetika_bes_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. syndetika Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II project</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h4><i>Glycine syndetika</i> accessions</h4>

<b>G1300</b> (Eastern Queensland region of Australia) <br/>
Glycine syndetika (B.E. Pfeil & Craven) is a twining perennial plant with three leathery, often persistent leaflets. Flowers are somewhat clustered towards to the top of the inflorescences and pods contain 4-9 relatively large square seeds (Pfeil. BE et al., 2006).

<br/><i>No resources are currently listed for accession <b>G1300</b></i>.





<hr/>

<h3 id="D3-tomentella"><i>Glycine D3-tomentella</i></h3>






<p>A complex of diploid and tetraploid taxa are lumped under the name "G. tomentella" but are each reproductively isolated species, e.g. G. tomentella D3 belongs to the D-genome, whereas D1 G. tomentella belongs to the E-genome.</p>

<p>NCBI taxonomy ID: <b>2908013</b></p>


<h4><i>Glycine D3-tomentella</i> resources</h4>
<dl>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_tomentella_BES_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a1 (Gmax1.01)</a></dt>
  <dd>GBrowse for G. tomentella Bac End Sequence alignments on Glyma.Wm82.a1 (Gmax1.01)</dd>
  
  <dt><a href="https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_tomentella_bes_alignments_relaxed_filtering">SoyMap2 Diversity Browser on Glyma.Wm82.a2 (Gmax2.0)</a></dt>
  <dd>GBrowse for G. tomentella Bac End Sequence alignments on Glyma.Wm82.a2 (Gmax2.0)</dd>
  
  <dt><a href="/projects/zhuang_wang_2022">SoyMap II</a></dt>
  <dd>SoyMap II project to sequence perennial relatives of soybean.</dd>
  
</dl>


<h4><i>Glycine D3-tomentella</i> accessions</h4>

<b>G1403</b> (Australian state of Queensland and the eastern portion of Papua New Guinea) <br/>
G. D3 tomentella is a diploid (2n = 40) member of the complex of diploid and tetraploid taxa lumped under the name 'G. tomentella'. G. D3 tomentella is distinguished from other diploid members of Glycine tomentella species complex by ovated to broadly lanceolate leaves, a seed coat or testa that is dark brown to black (to see the testa the persistent endocarp must be removed) and pods 1.5 to 2 cm long with 2 to 6 seeds per pod (Hill, 1999). G. D3 tomentella occurs in the Australian state of Queensland and the eastern portion of Papua New Guinea (Ratnaparkhe et al 2011; Gonzalez-Orozco et al., 2012).

<br/><i>No resources are currently listed for accession <b>G1403</b></i>.







  </div>

  

</div>


</body>
</html>
