---
layout: default
title: 2025 SBW Poster Winners
sitemap: community/sbw
---
<h2>2025 Soybean Breeders' Workshop Graduate Student Poster Awards</h2>

<div class="uk-margin-remove uk-grid-collapse uk-flex-center" uk-grid>
    <div class="uk-width-1-2@m">
        <div class="uk-margin-large-bottom uk-card uk-card-default">
            <div class="uk-card-media-top">
                <img src="https://data.soybase.org/annex/Glycine/max/meetings/soybean_breeders_workshop/SBW_2025/SBW2025_PosterWinners.jpeg"  alt="2025 Graduate Student Poster Winners">
            </div>
            <div class="uk-card-footer">
                <p>Left to Right: Jake Spittle, Rafael Gonçalves Marmo, Anser Mahmood</p>
            </div>
        </div>
    </div>
</div>


<div class="uk-card-header uk-text-bold uk-card-primary uk-padding-small">
	<h3>2025 Graduate Student Poster Winner's Abstracts</h3>
</div>

<br> 

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Rafael Gonçalves Marmo, University of Arkansas
        </a>
            <div class="uk-accordion-content">
                <p><b>Genomic Prediction to Optimize Soybean Grain Yield Stability in Multi-Environment Trials</b></p>
                <p> 
                    Historical breeding efforts to develop high-yielding soybean cultivars have resulted in subpar seed composition, mainly due to the strong negative correlation between grain yield and seed protein content. 
                    Breeders are now striving to develop soybean cultivars with improved seed composition and highly stable performance within their target environment. 
                    Despite these efforts, genotype-by-environment interaction (G×E) has been a barrier to achieving this goal. 
                    A total of 1,406 genotypes, ranging from maturity groups IV to V, were evaluated for grain yield in preliminary and final replicated trials conducted at three and five locations in Arkansas, respectively. 
                    Three stability methods—including the Superiority Index, Shukla's Stability Variance, and Wricke's Ecovalence—were applied using the R package "StatgenGxE.". 
                    A genomic best linear unbiased prediction (GBLUP) model was evaluated for superiority index using repeated five-fold cross-validation. 
                    The dataset was partitioned into training (80%) and testing (20%) sets. Genotypes have shown substantial differences in overall stability, ranging from 0.01 to 54.26 in Superiority Index, 0.01 to 25.37 in Shukla's Stability Variance, and 0.14 to 504.22 in Wricke's Ecovalence. 
                    Linoleic acid showed a significant negative correlation with Shukla's Stability Variance (-0.11) and Wricke's Ecovalence (-0.12), while seed protein content exhibited a significant negative correlation with Wricke's Ecovalence (-0.16). 
                    These results indicate that variable seed composition may affect soybean grain yield stability across environments. The GBLUP model achieved a mean accuracy of 0.62 and a mean RMSE of 5.24 during five-fold cross-validation. 
                    These results indicate that GBLUP effectively leverages genomic information to predict grain yield stability in multi-environment trials. 
                </p>
            </div>
    </li>
  <hr>
</ul>

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Jake Spittle, University of Georgia
        </a>
            <div class="uk-accordion-content">
                <p><b>Mining novel QTLs for insect resistance from wild soybean</b></p>
                <p>
			        In 2022, soybean [Glycine max] yield losses and management costs caused by insects  in the U.S. totaled over $1.4 billion. 
			        Control can be achieved by using genes for increased plant resistance to leaf-chewing insects, which can minimize the need for insecticide applications and protect yield. 
			        Wild soybean [G. soja] has untapped genetic diversity that can serve as a source of insect resistance for breeders to complement the resistance found in G. max. 
			        Thus, 112 accessions comprising the soja core collection were screened for resistance to leaf-chewing insects. 
			        Lack of preference was evaluated in a greenhouse using soybean loopers [Chrysodeixis includens], and through an open infestation of Mexican bean beetles [Epilachna varivestis] in the field. 
			        Two years of field data were gathered, and the second year of greenhouse data is currently being generated. G24-W751 was the most resistant accession in the field studies, averaging 5.7% defoliation. 
			        This accession is over twice as resistant as the most resistant Glycine max check, Benning EMGH, which contains four stacked QTLs for defoliator resistance and averaged 12.2% defoliation. 
			        Additionally, G24-W751 was also the most resistant accession in greenhouse testing performed thus far. 
			        Haplotypes deduced from SoySNP50k chip data suggest that G24-W751 may have the two largest insect resistance QTL, M and E, but no other known QTLs for defoliator resistance that would explain the enhanced resistance.  
			        A biparental mapping population is being developed to identify novel QTLs from this wild accession."
                </p>
            </div>
    </li>
  <hr>
</ul>

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Anser Mahmood, University of Missouri-Columbia 
        </a>
            <div class="uk-accordion-content">
                <p><b>Characterizing Novel Alleles for Effective Resistance Against Soybean Cyst Nematode</b></p>
                <p>
			        Soybean cyst nematode (SCN) is a destructive pathogen of soybean responsible for annual yield loss exceeding $1.5 billion in the United States. 
			        In this study, we aimed to identify novel resistance genes/alleles for several SCN populations and reveal allelic variation underlying identified candidate genes. 
			        To this end, we conducted genome-wide association studies (GWAS) on a panel of advanced breeding lines and cultivars from the University of Missouri’s northern and southern soybean breeding programs, as well as material evaluated in the USDA Uniform Soybean Trials – Northern Region. 
			        Additionally, we developed Soybean Structural Variant (SV) Navigator which visualizes short-read alignment files to enable identification of SVs lost in the variant-calling pipelines. 
			        Through GWAS, we identified quantitative trait nucleotides (QTNs) associated with SCN resistance on chromosomes 2, 8, 11, 14, 17, and 18. 
			        QTNs located on chromosomes 8, 11, and 18 matched previously identified Rhg4, Rhg2, and Rhg1 SCN resistance genes, respectively. Within QTN02, we were able to confirm a recently identified GmSNAP02 resistance gene and observe the allelic polymorphisms for this gene using Soybean SV Navigator. 
			        For QTN14, we determined GmSNAP14 as a candidate gene due to its proximity with the QTN. 
			        Using Soybean SV Navigator, we were able to identify two unique resistance alleles containing SVs for GmSNAP14. 
			        Overall, this study identifies novel alleles underlying resistance against virulent SCN populations. 
			        Furthermore, we provide a tool to identify SVs from short-read alignment data. 
			        Insights from this study help understand the landscape of SCN resistance and elucidate the gene combinations required for resistance.
                </p>
            </div>
    </li>
  <hr>
</ul>
