---
layout: default
title: 2024 SBW Poster Winners
sitemap: community/sbw
---
<h2>2024 Soybean Breeders' Workshop Graduate Student Poster Awards</h2>

<div class="uk-card uk-card-default">
    <div class="uk-card-media-top">
        <img src="https://data.soybase.org/annex/Glycine/max/meetings/soybean_breeders_workshop/SBW_2024/SBW2024_Poster_Winners.jpg"  alt="2024 Graduate Student Poster Winners">
    </div>
    <div class="uk-card-body">
        <h3 class="uk-card-title">Left to Right: Kieth Rincker (Committee Member), Kelsey Boucher, Mary Jane Espina, Forrest Hanson, Cleiton Wartha (Committee Member)
        </h3>
    </div>
</div>

<br>
<br>

<div class="uk-card-header uk-text-bold uk-card-primary uk-padding-small">
	<h3>2024 Graduate Student Poster Winner's Abstracts</h3>
</div>

<br> 

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Mary Jane Espina, University of Minnesota
        </a>
            <div class="uk-accordion-content">
                <p>This work provides the soybean community with an updated telomere-to-telomere genome assembly (version Wm82.a6) of the soybean cultivar Williams 82. Williams 82 is known to have areas of genomic heterogeneity among different sub-lines. Thus, the Wm82.a6 assembly is derived from a specific sub-line known as Williams 82-ISU-01. The genome was assembled using Pacific BioSciences HiFi reads and integrated into chromosomes using HiC. The  Wm82.a6 has 1.01 Gb assembly size, 48,387 predicted gene models, and comprises a total of 36 contigs. Comparisons of Wm82.a6 with other near-gapless assemblies of Williams 82 reveal both known and previously unknown regions of chromosomal heterogeneity, including distinct haplotype differences on chromosomes 03, 07 and 12. This includes a previously characterized region of differential introgression from the genotype 'Kingwa' on segments of chromosome 3. Our analysis revealed a previously unknown large (~20 Mb)heterogeneous region on the pericentromeric region of chromosome 12; Wm82.a6 matches the 'Williams' haplotype while the other two completed assemblies do not match the haplotype of either parent of Williams 82.  In addition to Wm82.a6 assembly, we also assembled the genome of soybean line 'Fiskeby III', a rich resource for abiotic stress resistance genes. We used these assemblies to study the genomic variation between 'Fiskeby III' and the Wm82.a6 reference within a fine-mapped QTL for iron deficiency chlorosis resistance, revealing candidate sequence polymorphisms that may be underlying the QTL variation. The Wm82.a6 and 'Fiskeby III' genomes described here will enhance comparative and functional genomics capacities and applications in the soybean community.
                </p>
            </div>
    </li>
  <hr>
</ul>

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Kelsey Boucher, University of Guelph
        </a>
            <div class="uk-accordion-content">
                <p>Sudden Death Syndrome (SDS) is the third most damaging soybean disease in the United States and the second most damaging in Southwestern Ontario, causing the loss of millions of bushels each year. The disease, caused by Fusarium virguliforme first appeared in Kent County, Ontario, in 1993. The disease has since spread throughout Southern Ontario and increased in severity, decimating expected yields in infected fields. This study aims to evaluate the University of Guelph genomic diversity panel consisting of 250 soybean genotypes for SDS resistance. The panel consists mostly of historic and modern cultivars from several breeding programs in Ontario and Quebec in the range of maturities from 00 to 2, including those that are resistant to Soybean Cyst Nematodes (SCN), which has been reported in some studies as being correlated to SDS resistance. It is hypothesized that the University of Guelph GWAS panel includes genotypes covering a range of resistant to susceptible reactions to SDS, despite the absence of a targeted breeding effort for SDS applied by various programs. The primary objective of this study is to employ a GWAS to identify MTAs for resistance and rate the leaf and root rot symptoms of cultivars when grown in infected conditions. SDS resistance was evaluated through three field site evaluations and growth chamber inoculation. Field evaluations have provided foliar symptom data, while the ongoing growth chamber inoculations will evaluate root rot symptoms exhibited by each genotype. Preliminary field data showed a range of symptoms from 0 to 100 (indicating premature death), with 56 varieties receiving an average score between 0 and10 (highly tolerant), 104 between 11 and 30 (moderately tolerant), 86 between 31 and 60 (moderately susceptible) and 4 between 61 and 100 (highly susceptible). These results indicate the panel' genetic diversity for SDS resistance as hypothesized. GWAS will be conducted using the phenotypic data from both evaluations to identify and compare MTAs for both the foliar and root symptoms. 
                </p>
            </div>
    </li>
  <hr>
</ul>

<ul uk-accordion>
    <li class="ukaccordion">
        <a class="uk-accordion-title"> 
            Forrest Hanson, North Dakota State University
        </a>
            <div class="uk-accordion-content">
                <p>Soybean is a relatively new crop in North Dakota; however, it has become the number one crop in the state for acres planted and production value. Public breeding efforts began in 1986 through North Dakota State University, and during this time 39 cultivars in maturity groups 00 and 0 have been released. Although yields have increases during this time, the yield gains and genetic diversity of the program have not been studied. It is important to understand these components of a breeding program to ensure yields do not become stagnant. We would like to determine the amount of genetic diversity in this program to improve yields further. This knowledge will allow our program and other breeders in the maturity group 00 and 0 environment to continue to improve yields. 
                </p>
            </div>
    </li>
  <hr>
</ul>