<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Arachis &#8211; LIS: Legume Information System</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="A collaborative, community resource to facilitate crop improvement by integrating genetic, genomic, and trait data across legume species.">
  <link rel="manifest" type="application/manifest+json; charset=utf-8" href="/manifest.json" />
  <meta name="robots" content="all">
  <meta name="author" content="">
  
  
  <link rel="canonical" href="/taxa/arachis/">
  <link rel="alternate" type="application/rss+xml" title="RSS Feed for LIS: Legume Information System" href="/feed.xml" />

  <!-- Custom CSS -->
  <link rel="stylesheet" href="/assets/css/uikit.min.css" type="text/css" />
  <link rel="stylesheet" href="/assets/css/theme.css" type="text/css" />

  <!-- place globally-included stylesheet content here -->


  <!-- Custom JS -->
  <script src="/assets/js/uikit.min.js"></script>
  <script src="/assets/js/uikit-icons.min.js"></script>

  <!-- place globally-included scripts here -->


  <!-- Icons -->
  <link rel="apple-touch-icon" sizes="57x57" href="/assets/icons/apple-touch-icon-57x57.png">
  <link rel="apple-touch-icon" sizes="114x114" href="/assets/icons/apple-touch-icon-114x114.png">
  <link rel="apple-touch-icon" sizes="72x72" href="/assets/icons/apple-touch-icon-72x72.png">
  <link rel="apple-touch-icon" sizes="144x144" href="/assets/icons/apple-touch-icon-144x144.png">
  <link rel="apple-touch-icon" sizes="60x60" href="/assets/icons/apple-touch-icon-60x60.png">
  <link rel="apple-touch-icon" sizes="120x120" href="/assets/icons/apple-touch-icon-120x120.png">
  <link rel="apple-touch-icon" sizes="76x76" href="/assets/icons/apple-touch-icon-76x76.png">
  <link rel="apple-touch-icon" sizes="152x152" href="/assets/icons/apple-touch-icon-152x152.png">
  <link rel="apple-touch-icon" sizes="180x180" href="/assets/icons/apple-touch-icon-180x180.png">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-192x192.png" sizes="192x192">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-160x160.png" sizes="160x160">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-96x96.png" sizes="96x96">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-16x16.png" sizes="16x16">
  <link rel="icon" type="image/png" href="/assets/icons/favicon-32x32.png" sizes="32x32">
  <link rel="shortcut icon" href="/assets/favicon.ico">
</head>


<body>




<div class="uk-navbar-container tm-navbar-container uk-sticky uk-sticky-fixed" uk-sticky="media: 960" style="position: fixed; top: 0px; width: 1199px;">
  <div class="uk-container uk-container-expand">
    <nav class="uk-navbar">
      <div class="uk-navbar-left">
        <a class="uk-navbar-item uk-logo" href="/">
          
          <img class="uk-margin-small-right" src="/assets/img/logo.png" width="47" height="47" />
          
          <div>
            LIS: Legume Information System
            
            <div class="uk-navbar-subtitle">Information about legume traits for crop improvement</div>
            
          </div>
        </a>
      </div>
      <div class="uk-navbar-right">
        <ul class="uk-navbar-nav uk-visible@m">

        <li><a href="/">Home</a></li>
<li>
  <a class="taxon_list" href="/taxa">Taxa</a>
  
  
  <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="toggle: .taxon_list; delay-hide: 0;">
    <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
    
      
        
          <li> <a href="/taxa//arachis"> Arachis (peanut: domesticated and wild)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//cajanus"> Cajanus (pigeonpea)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//cicer"> Cicer (chickpea)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//glycine"> Glycine (soybean)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//lotus"> Lotus (birdsfoot trefoil; model species)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//lupinus"> Lupinus (several species of lupin grain legumes)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//medicago"> Medicago (alfalfa/lucerne, barrel medic; model species)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//phaseolus"> Phaseolus (common bean, lima bean, tepary bean, others)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//pisum"> Pisum (pea)</a></li>
        
      
    
      
        
          <li> <a href="/taxa//trifolium"> Trifolium (clovers)</a></li>
        
      
    
      
        
      
    
      
        
      
    
      
        
      
    
      
        
      
    
      
        
      
    
      
        
      
    
      
        
      
    
    <li><a href="/taxa">OTHERS (Many special-focus species and collections)</a></li>
    </ul>
  </div>
  
</li>
<li>
  <a class="tools" href="/tools">Tools</a>
  
  
  <div class="uk-navbar-dropdown uk-width-auto" uk-dropdown="toggle: .tools; delay-hide: 0;">
    <ul class="uk-nav uk-navbar-dropdown-nav uk-link-reset">
    
      
        <li><a href="#">Gene Families</a></li>
      
        <li><a href="#">Maps</a></li>
      
        <li><a href="#">QTL Search</a></li>
      
        <li><a href="#">Gene Search</a></li>
      
        <li><a href="#">Protein Domains</a></li>
      
        <li><a href="#">Genome Context Viewer</a></li>
      
        <li><a href="#">Data Store</a></li>
      
        <li><a href="#">Legume Mines</a></li>
      
        <li><a href="#">DSCensor Data Overviews</a></li>
      
        <li><a href="#">Expression</a></li>
      
        <li><a href="#">ZZBrowse GWAS/QTL</a></li>
      
    
      
        <li><a href="#">BLAST Sequence Search</a></li>
      
        <li><a href="#">BLAT Sequence Search</a></li>
      
        <li><a href="#">Keyword Search</a></li>
      
        <li><a href="#">SequenceServer</a></li>
      
        <li><a href="#">Annotate Your Sequences</a></li>
      
    
    </ul>
  </div>
  
</li>
<li><a href="/download">Download</a></li>
<li><a href="/community">Community</a></li>
<li><a href="/contact">Contact</a></li>
<li><a href="/help">Help</a></li>


        
          <li><a class="tm-no-underline" href="https://twitter.com/legumefed" target="_blank" title="Follow us on Twitter"><span uk-icon="twitter"></span></a></li>
        
        
          <li><a class="tm-no-underline" href="" target="_blank" title="Subscribe to our newsletter"><span uk-icon="mail"</span></a></li>
        
        
          <li><a class="tm-no-underline" href="https://github.com/legumeinfo" target="_blank" title="Fork us on GitHub"><span uk-icon="github"></span></a></li>
        
        </ul>
      </div>
    </nav>
  </div>
</div>



  <div class="tm-sidebar-left uk-visible@m">
  <h3>Tools</h3>
  
  
  
  <ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
    <li class="uk-nav-header">Browse and Search</li>
    
    <li><a href="#">Gene Families</a></li>
    
    <li><a href="#">Maps</a></li>
    
    <li><a href="#">QTL Search</a></li>
    
    <li><a href="#">Gene Search</a></li>
    
    <li><a href="#">Protein Domains</a></li>
    
    <li><a href="#">Genome Context Viewer</a></li>
    
    <li><a href="#">Data Store</a></li>
    
    <li><a href="#">Legume Mines</a></li>
    
    <li><a href="#">DSCensor Data Overviews</a></li>
    
    <li><a href="#">Expression</a></li>
    
    <li><a href="#">ZZBrowse GWAS/QTL</a></li>
    
  </ul>
  
  <ul class="uk-nav uk-nav-default tm-nav uk-margin-top">
    <li class="uk-nav-header">Search sequences and features against sequence databases</li>
    
    <li><a href="#">BLAST Sequence Search</a></li>
    
    <li><a href="#">BLAT Sequence Search</a></li>
    
    <li><a href="#">Keyword Search</a></li>
    
    <li><a href="#">SequenceServer</a></li>
    
    <li><a href="#">Annotate Your Sequences</a></li>
    
  </ul>
  
  
</div>



<div class="tm-main uk-section uk-section-default">
  <div class="uk-container uk-container-small uk-position-relative">
    

<h2><i>Arachis</i></h2>

<h3>Tools and resources for the genus as a whole</h3>


<dt>

  
    <dd><strong><i>Arachis</i></strong> (warm-season oilseed crop, indigenous to South America.)</dd>
  
  

  
  
    <dd><a href="https://mines.legumeinfo.org/peanutmine/begin.do">PeanutMine</a> (InterMine interface for accessing genetic and genomic data for several species in Arachis.)</dd>
  
    <dd><a href="https://zzbrowse.legumeinfo.org/?tab=WhGen&datasets=Peanut%20GWAS&chr=Arahy.01&selected=100000&window=250000&datasets2=Soybean%20GWAS&chr2=Gm01&selected2=100000&window2=250000">ZZBrowse</a> (Association viewers (QTL, GWAS))</dd>
  
    <dd><a href="https://peanutbase.org/germplasm/gigwa">Gigwa</a> (Genetic variation viewer )</dd>
  
    <dd><a href="https://peanutbase.org/germplasm/gcvit">GCViT</a> (Genetic variation viewer )</dd>
  
    <dd><a href="https://legumeinfo.org/gcv2/gene;lis=arahy.P7Z3M8?q=arahy.CMRI1Q&sources=lis&algorithm=repeat&match=10&mismatch=-1&gap=-1&score=30&threshold=25&bmatched=20&bintermediate=10&bmask=10&linkage=average&cthreshold=20&neighbors=10&matched=4&intermediate=5&bregexp=&border=chromosome&regexp=&order=distance">Genome Context Viewer</a> (Browser for dynamically discovering and viewing genomic synteny across selected species.)</dd>
  
    <dd><a href="https://legumeinfo.org/germplasm/map#?zoom=5&maxRecs=1000&taxonQuery=Arachis&traitOverlay=&traitScale=global&country=&geocodedOnly&traitExcludeUnchar=false&limitToMapExtent=false&lat=-15.71&lng=-62.62&mapHeight=430&baseMap=ESRI%20-%20NatGeo%20(default,%20reference%20map)&ne_lat=-6.446317749457646&ne_lng=-38.3203125&sw_lat=-24.56710835257599&sw_lng=-86.923828125&accessionIdsInclusive=false">Germplasm GIS</a> (Geographic information system viewer, showing collection locations for Arachis data held by the U.S. National Plant Germplasm System.)</dd>
  

</dt>

<h3>Tools and resources for particular species</h3>


<dt>

  
    <dd><strong>Arachis hypogaea</strong> (peanut)</dd>
    
      <img src="/assets/img/species_images/arahy.jpg" alt="Image: arahy">
    
    <dd>Peanut, <i>Arachis hypogaea</i>, was domesticated at the foot of the South American Andes within modern-day Argentina, Bolivia, or Paraguay.  Cultivated peanuts are allotetraploids, with sub-genomes probably contributed by the hybridization of the diploid species <i>Arachis duranensis</i> and <i>Arachis ipaensis</i>.</dd>
    <dd></dd>
    
      <dd><u>Genome browsers and genomic data</u></dd>
      
        <dd>Tifrunner (Arachis hypogaea accession Tifrunner)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/">Download from Data Store</a>;
            
          </dd>
        
      
        <dd>Tifrunner (Arachis hypogaea accession Tifrunner)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm2:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/">Download from Data Store</a>;
            
          </dd>
        
      
        <dd>Fuhuasheng (Arachis hypogaea accession Fuhuasheng)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/">Download from Data Store</a>;
            
          </dd>
        
      
        <dd>Shitouqi (Arachis hypogaea accession Shitouqi)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/">Download from Data Store</a>;
            
          </dd>
        
      
    
    <br>
  
    <dd><strong>Arachis duranensis</strong> (Arachis duranensis / wild peanut)</dd>
    
      <img src="/assets/img/species_images/aradu.jpg" alt="Image: aradu">
    
    <dd><i>Arachis duranensis</i>, a wild herb native to Argentina, Bolivia, and Paraguay, is one of the diploid progenitors of cultivated peanut. Its genome was sequenced as part of the Peanut Genomics Initiative, to help in the accurate assembly of the more important cultivated peanut, <i>A. hypogaea</i>. Cultivated peanut derived all of its genetic material from two wild ancestors, <i>A. duranensis</i> and <i>A. ipaensis</i> whose genomes merged several thousand years ago, in a rare genetic event. The wild species, therefore, have half as much genetic material as cultivated peanut. These two “simpler” genomes have first been sequenced toward achieving its ultimate goal: the complete genomic sequence for cultivated peanut. The cultivated peanut is an allotetraploid (2n=4x=40) that carries both the A and B genomes and <i>A. duranensis</i> (2n=2x=20) has likely contributed the A genome, the smaller set of chromosomes in the karyotype. These two ancestral diploids separated from each other about 3 million years ago. The genome merger, allopolyploidy event, ocurred  relatively very recently, five to ten thousand years ago, followed by domestication in South America from where it appeared in most part of the world by 1600.</dd>
    <dd></dd>
    
      <dd><u>Genome browsers and genomic data</u></dd>
      
        <dd>V14167 (Arachis duranensis accession V14167)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/duranensis/">Download from Data Store</a>;
            
          </dd>
        
      
    
    <br>
  
    <dd><strong>Arachis ipaensis</strong> (Arachis duranensis / wild peanut)</dd>
    
      <img src="/assets/img/species_images/araip.jpg" alt="Image: araip">
    
    <dd><i>A. ipaensis</i>, native to Bolivia, is one of the diploid ancestors of cultivated peanut. The genome of <i>A. ipaensis</i> was sequenced as part of the Peanut Genomics Initiative, to help in the accurate assembly of the tetraploid domestic peanut, <i>A. hypogaea</i>. Cultivated peanut derived all of its genetic material from two wild ancestors, <i>A. duranensis</i> and <i>A. ipaensis</i> whose genomes merged several thousand years ago, in a rare genetic event.  The wild species, therefore, have half as much genetic material as cultivated peanut.  These two “simpler” genomes have first been sequenced toward achieving its ultimate goal: the complete genomic sequence for cultivated peanut. The cultivated peanut is an allotetraploid (2n=4x=40) that carries both the A and B genomes and <i>A. ipaensis</i> (2n=2x=20) has likely contributed the B genome, the larger set of chromosomes in the karyotype. These two ancestral diploids separated from each other about 3 million years ago.  The genome merger, allopolyploidy event, ocurred  relatively very recently, five to ten thousand years ago, followed by domestication in South America from where it appeared in most part of the world by 1600.</dd>
    <dd></dd>
    
      <dd><u>Genome browsers and genomic data</u></dd>
      
        <dd>K30076 (Arachis ipaensis accession K30076)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">Download from Data Store</a>;
            
          </dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm2:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="#">JBrowse at JGI</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/ipaensis/">Download from Data Store</a>;
            
          </dd>
        
      
    
    <br>
  
    <dd><strong>Arachis cardenasii</strong> (Mani silvestre (wild peanut))</dd>
    
    <dd><i>Arachis cardenasii</i> Krapov. & W.C. Greg. is a wild peanut relative native to Bolivia. It has been used by peanut breeders and geneticists in interspecific hybrids with cultivated peanut (<i>A. hypogaea</i>) as a source of pest and disease resistance. A. cardenasii is a diploid  species in the A-genome group of <i>Arachis</i>, similar to <i>A. duranensis</i>, which contributed one of the two subgenomes to tetraploid cultivated peanut.</dd>
    <dd></dd>
    
      <dd><u>Genome browsers and genomic data</u></dd>
      
        <dd>K10017 (Arachis cardenasii, accession K10017)</dd>
        
          <dd>&nbsp; &nbsp;
            Assembly gnm1:
          </dd>
          <dd>&nbsp; &nbsp;
            
              <a href="#">GBrowse at LIS</a>;
            
              <a href="#">JBrowse at LIS</a>;
            
              <a href="https://peanutbase.org/data/v2/Arachis/cardenasii/">Download from Data Store</a>;
            
          </dd>
        
      
    
    <br>
  

</dt>

  </div>
</div>

</body>
</html>
