---
layout: default
title: Genome and Annotation Information
sitemap: sequence map
tools_menu: true

wm82_a1_details:
- accession: "GCF_000004515.3"
  url: "https://www.ncbi.nlm.nih.gov/assembly/GCF_000004515.3/"
  source: "RefSeq"
- accession: "GCA_000004515.2"
  url: "https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000004515.3/"
  source: "GenBank"
- accession: "<i>Glycine max</i> v1.1"
  url: "https://data.jgi.doe.gov/search?q=Glycine+max&expanded=Phytozome-508"
  source: "JGI/Phytozome"
- accession: "Wm82.gnm1"
  url: "https://data.soybase.org/Glycine/max/genomes/Wm82.gnm1.FCtY/"
  source: "SoyBase/LIS Data Store"

wm82_a2_details:
- accession: "GCF_000004515.5"
  url: "https://www.ncbi.nlm.nih.gov/assembly/GCF_000004515.5/"
  source: "RefSeq"
- accession: "GCA_000004515.2"
  url: "https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000004515.5/"
  source: "GenBank"
- accession: "<i>Glycine max</i> Wm82.a2.v1"
  url: "https://phytozome-next.jgi.doe.gov/info/Gmax_Wm82_a2_v1"
  source: "JGI/Phytozome"
- accession: "Wm82.gnm2"
  url: "https://data.soybase.org/Glycine/max/genomes/Wm82.gnm2.DTC4/"
  source: "SoyBase/LIS Data Store"

wm82_a4_details:
- accession: "GCF_000004515.5"
  url: "https://www.ncbi.nlm.nih.gov/assembly/GCF_000004515.5/"
  source: "RefSeq"
- accession: "GCA_000004515.2"
  url: "https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000004515.5/"
  source: "GenBank"
- accession: "<i>Glycine max</i> Wm82.a2.v1"
  url: "https://phytozome-next.jgi.doe.gov/info/Gmax_Wm82_a2_v1"
  source: "JGI/Phytozome"
- accession: "Wm82.gnm2"
  url: "https://data.soybase.org/Glycine/max/genomes/Wm82.gnm2.DTC4/"
  source: "SoyBase/LIS Data Store"

fiskebyiii_a1_details:
- accession: "<i>Glycine max</i> Fiskeby III v1"
  url: "https://phytozome-next.jgi.doe.gov/info/GmaxFiskeby_v1_1"
  source: "JGI/Phytozome"
- accession: "FiskebyIII.gnm1"
  url: "https://data.soybase.org//Glycine/max/genomes/FiskebyIII.gnm1.F177/"
  source: "SoyBase/LIS Data Store"

lee_a1_details:
- accession: "GCA_002905335.1"
  url: "https://ncbi.nlm.nih.gov/assembly/GCA_002905335.1/"
  source: "GenBank"
- accession: "<i>Glycine max</i> Lee.v1.1"
  url: "https://phytozome-next.jgi.doe.gov/info/GmaxLee_v1_1"
  source: "JGI/Phytozome"
- accession: "Lee.gnm1"
  url: "https://data.soybase.org//Glycine/max/genomes/FiskebyIII.gnm1.F177/"
  source: "SoyBase/LIS Data Store"

---

<H2><b>Genome Assembly Page</b></H2>

<p>
Since the release of the first full soybean genome assembly in 2010, assemblies have been generated for more than 50 accessions, including multiple assemblies for the first reference, Williams 82 (Wm82).
</p>

<p>
There are several nomenclature patterns for the assemblies and annotations. The pattern used by the DOE-JGI and SoyBase has generally taken the form Wm82.a4.v1, with the middle field ("a4") indicating assembly version and the last field (v1) indicating the annotation version. Within the SoyBase and LegumeInfo Data Store, the pattern takes the form Wm82.gnm4.ann1 -- again, with the middle field ("gnm4") indicating assembly version and the last field (ann1) indicating the annotation version.
</p>

<p>
See below for details about the main reference genotypes and assemblies in use in the U.S.
</p>


<H3><b>Williams 82 Reference Genome Assemblies</b></H3>

<p>
The soybean Williams 82 (Wm82) reference genome has been released three times since its first release as a whole-genome shotgun approach in 2010.
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Assembly - Common</b></th>
      <th><b>Assembly - Data Store</b></th>
      <th><b>Annotation - Common</b></th>
      <th><b>Annotation - Data Store</b></th>
      <th><b>Annotation - JGI</b></th>
    </tr>
  </thead>
  <tbody>
    {% for name in site.data.genome-info-tables.wm82_versions %}
      <tr>
        <td>{{ name.assembly_common }}</td>
        <td>{{ name.assembly_ds }}</td>
        <td>{{ name.annot_common }}</td>
        <td>{{ name.annot_ds }}</td>
        <td>{{ name.annot_jgi }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H3><b>Wm82 Genome Assembly Details</b></H3>

<p>
The soybean Williams 82 (Wm82) reference genome has been released three times since its first release as a 8X whole-genome shotgun approach in 2010. The soybean community has adopted naming standards to accommodate multiple Glycine species, culivars, and genome assemblies.
</p>


<H4><b>Wm82.a1.v1.1</b></H4>

<p>
Originally generated by the Joint Genome Institute (JGI) in 2008 as a 8X whole-genome shotgun assembly. The assembly was officially reported in 2010 in Nature. This assembly is known as Wm82.a1.v1.1 at SoyBase. It differs from Wm82.a1.v1 by the inclusion of more expression data used to construct the gene models.
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Accession</b></th>
      <th><b>Source</b></th>
    </tr>
  </thead>
  <tbody>
    {% for item in page.wm82_a1_details %}
      <tr>
        <td><a href="{{ item.url }}">{{ item.accession }}</a></td>
        <td>{{ item.source }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H4><b>Wm82.a2.v1</b></H4>

<p>
The original assembly (Glyma1) was generated by the Joint Genome Institute (JGI) in 2006 and corrected by including more EST information as well as a denser genetic map. This assembly (Wm82.a2.v1) replaces the Glyma1 assembly (Wm82.a1.v1.1) by including the data from a genetic map produced by Perry Cregan and Qijian Song at the Beltsville Agricultural Research Center West, USDA, ARS (<a href="https://doi.org/10.1186/s12864-015-2344-0">Song Q, Jenkins J, Jia G, et al. BMC Genomics. 2016;17:33</a>). This corrects several issues in pseudomolecule reconstruction in the Glyma1 assembly. The Wm82.a2.v1 gene set was created by integrating ~1.6 million ESTs, some 454 ESTs and 1.5 billion paired-end Illumina RNA-seq reads with homology-based gene predictions. It is the version known as Wm82.a2.v1 at SoyBase
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Accession</b></th>
      <th><b>Source</b></th>
    </tr>
  </thead>
  <tbody>
    {% for item in page.wm82_a2_details %}
      <tr>
        <td><a href="{{ item.url }}">{{ item.accession }}</a></td>
        <td>{{ item.source }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H4><b>Wm82.a4.v1</b></H4>

<p>
Originally generated by the Joint Genome Institute (JGI) in 2019. This version differs from Wm82.a2.v1 by the inclusion of more RNA evidence for the new gene models and the inclusion of optical mapping to resolve structural variations between other assemblies. The assembly was further enhanced by the use of long PACBIO reads. This is the version known as Wm82.a4.v1 at SoyBase.
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Accession</b></th>
      <th><b>Source</b></th>
    </tr>
  </thead>
  <tbody>
    {% for item in page.wm82_a4_details %}
      <tr>
        <td><a href="{{ item.url }}">{{ item.accession }}</a></td>
        <td>{{ item.source }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H3><b>Other Cultivar Genome Assembly Details</b></H3>


<H4><b>CV Fiskeby III Assembly Details</b></H4>

<p>
The soybean cultivar Fiskeby III (PI 438471) reference genome was released in 2020 by the Joint Genome Institue (JGI). The plant introduction was developed in Sweden in the early 1970's. It has been a focus of development because of its tolerance to many biotic and abiotic stresses.
</p>

<H4><b>Glyma.FiskebyIII.a1.v1.1</b></H4>

<p>
The Joint Genome Institute (JGI) released in the genome sequence of Fiskeby III in 2020. It is known as Glyma.FiskebyIII.a1.v1.1 at SoyBase.
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr class="uk-width-9-10 uk-text-bold">
      <th><b>Accession</b></th>
      <th><b>Source</b></th>
    </tr>
  </thead>
  <tbody>
    {% for item in page.fiskebyiii_a1_details %}
      <tr>
        <td><a href="{{ item.url }}">{{ item.accession }}</a></td>
        <td>{{ item.source }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H3><b>CV Lee Genome Assembly Details</b></H3>

<p>
The soybean cultivar Lee reference genome was produced by the Joint Genome Institute (JGI). The genome sequence was released in 2018. The cultivar Lee is a representative of the Southern soybean germplasm where Williams 82 is a representative of the Northern germplasm. The Southern germplasm has been under development for different biotic and abiotic phenotypes than that of the Northern germplasms and thus an examination of genome sequence of Lee in comparison with the genome sequence of Williams 82 may shed light on the source of the adaptive phenotypes in the Southern germplasm.
</p>

<H4><b>Glyma.Lee.a1.v1.1</b></H4>

<p>
Originally generated by the Joint Genome Institute (JGI) in 2018 where it is known as <i>Glycine max</i> Lee v1.1. This assembly is known as Glyma.Lee.a1.v1.1 at SoyBase.
</p>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Accession</b></th>
      <th><b>Source</b></th>
    </tr>
  </thead>
  <tbody>
    {% for item in page.lee_a1_details %}
      <tr>
        <td><a href="{{ item.url }}">{{ item.accession }}</a></td>
        <td>{{ item.source }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>

<H2><b>Assembly Chromosome Lengths</b></H2>

<div class="uk-overflow-auto">
<table class="uk-table uk-table-small uk-table-hover uk-table-divider uk-table-striped">
  <thead>
    <tr>
      <th><b>Chr.</b></th>
      <th><b>Wm82.a1</b></th>
      <th><b>Wm82.a2</b></th>
      <th><b>Wm82.a4</b></th>
      <th><b>Glyma.Lee.a1</b></th>
      <th><b>Glyma.ZH13.a1</b></th>
      <th><b>Glyma.FiskebyIII.a1</b></th>
    </tr>
  </thead>
  <tbody>
    {% for column in site.data.genome-info-tables.chr_lengths %}
      <tr>
        <td>{{ column.chr }}</td>
        <td>{{ column.wm82_a1 }}</td>
        <td>{{ column.wm82_a2 }}</td>
        <td>{{ column.wm82_a4 }}</td>
        <td>{{ column.glyma_lee_a1 }}</td>
        <td>{{ column.glyma_zh13_a1 }}</td>
        <td>{{ column.glyma_fiskebyiii_a1 }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
</div>
