---
layout: default
iframe_url: "https://gcvit.soybase.org"
redirect_from: /gcvit/
---
          
<div>
<h2 class="uk-text-bold">GCViT (Genotype Comparison Visualization Tool)</h2>
<p class="uk-text-bold"> <a href='https://youtu.be/B2gPVUipWo0'>Video Tutorial on GCViT</a> </p>

<!-- ################# add details button for Datasets on GCVIT-->
<p class="uk-text-bold">
  Learn more about the Datasets available on GCViT
</p>

  <button class="uk-button uk-button-primary uk-button-small" type="button" uk-toggle="target: #whole-plant-grouth-terms; animation: uk-animation-fade; cls: uk-hidden" onclick="rotateTriangle(this)" value="triangle-icon-whole-plant-growth"><div class="uk-flex">details<span id="triangle-icon-whole-plant-growth" uk-icon="icon:triangle-right" class="toggle-top-margin"></span></div></button>
  <div id="whole-plant-grouth-terms" class="uk-margin-small uk-hidden">
<!--<h3>  GCViT Datasets</h3>-->
<p>SNPs were sampled at a density of no more than one SNP per 25kb prior to display. This has the effect of partially normalizing the SNP density (though regions of the genome with very sparse SNP density will still show sparse density). </p>
<p><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Song_Hyten_2015/glyma.Wm82.gnm2.div.Song_Hyten_2015.PIconversion.txt.gz' class="uk-text-bold">Click Here To Download PI to Common Name Conversion Table</a></p>
<p style="color: blue"> If you would like to submit your data to be run on this tool please <span><a href='/about/' style="color: rgb(232, 0, 0)" class="uk-text-bold">Contact Us </a></span></p>

<table class="uk-table uk-table-small uk-table-hover uk-table-responsive uk-table-divider uk-table-striped">
 <thead>
  <tr>
    <th>Dataset</th>
    <th>Citation*</th> 
    <th>Source</th>
  </tr>
 </thead>
  <tr>
    <td>SoySNP50K</td>
    <td>Song, Q., Hyten, D. L., Jia, G., Quigley, C. V., Fickus, E. W., Nelson, R. L., Cregan, P. B. (2015). Fingerprinting soybean germplasm and its utility in genomic research. G3: Genes, Genomes, Genetics, 5(10). <a href='https://doi.org/10.1534/g3.115.019000'>doi.org/10.1534/g3.115.019000</a></td>
    <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Song_Hyten_2015/'>View Here</a></td>
  </tr>
  <tr>
    <td>USB 481 </td>
    <td>Valliyodan, B., Brown, A., Wang, J., Patil, G., Liu, Y., Otyama, P., Nelson, R., Vuong, T., Song, Q., Musket, T., Wagner, R., Marri, P., Reddy, S., Sessions, A., Wu, X., Grant, D., Bayer, P., Roorkiwal, M.,Varshney, R., Liu, X., Edwards, D., Xu, D., Joshi, T., Cannon, S., Nguyen, H. (2021). Genetic variation among 481 diverse soybean accessions, inferred from genomic re-sequencing. Sci. Data, 8;8(1):50 (2021). <a href='https://doi.org/10.1038/s41597-021-00834-w'>doi.org/10.1038/s41597-021-00834-w</a></td>
    <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Valliyodan_Brown_2021/'>View Here</a></td>
    </tr>
 <tr>
   <td>102 Canadian Accessions </td>
   <td>Torkamaneh, D., Laroche, J., Tardivel, A., O'Donoughue, L., Cober, E., Rajcan, I., Belzile, F. (2017). Comprehensive description of genomewide nucleotide and structural variation in short-season soya bean. Plant Biotechnology Journal. <a href='https://doi.org/10.1111/pbi.12825'>doi.org/10.1111/pbi.12825</a></td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Torkamaneh_Laroche_2017/'>View Here</a></td>
 </tr>
 <tr>
   <td>SoyNAM parents and Progeny </td>
   <td> Bandillo, Nonoy B., Aaron J. Lorenz, George L. Graef, Diego Jarquin, David L. Hyten, Randall L. Nelson, and James E. Specht. Genome-wide association mapping of qualitatively inherited traits in a germplasm collection. The plant genome 10, no. 2 (2017). <a href='https://doi.org/10.3835/plantgenome2016.06.0054'>doi.org/10.3835/plantgenome2016.06.0054</a><br/>
 Song, Qijian, Long Yan, Charles Quigley, Brandon D. Jordan, Edward Fickus, Steve Schroeder, Bao-Hua Song et al. Genetic characterization of the soybean nested association mapping population. The plant genome (2017). <a href='https://doi.org/10.3835/plantgenome2016.10.0109'>doi.org/10.3835/plantgenome2016.10.0109</a><br/>
 Diers, Brian W., Jim Specht, Katy Martin Rainey, Perry Cregan, Qijian Song, Vishnu Ramasubramanian, George Graef et al. Genetic Architecture of Soybean Yield and Agronomic Traits. G3: Genes, Genomes, Genetics 8, no. 10 (2018): 3367-3375. <a href='https://doi.org/10.1534/g3.118.200332'>doi.org/10.1534/g3.118.200332</a></td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Song_Yan_2017/'>View Here</a></td>
 </tr>
 
 <tr>
   <td>222 Korean Accessions using Axiom SoyaSNP array</td>
   <td> Lee, Y. G., Jeong, N., Kim, J. H., Lee, K., Kim, K. H., Pirani, A., ...Jeong, S.C. (2015). Development, validation and genetic analysis of a large soybean SNP genotyping array. The Plant Journal, 81(4), 625-636. <a href='https://doi.org/10.1111/tpj.12755'>doi.org/10.1111/tpj.12755</a></td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Lee_Jeong_2015/'>View Here</a></td>
 </tr>

 <tr>
   <td>4234 Korean Accessions using Axiom SoyaSNP array </td>
   <td>Jeong, S. C., Moon, J. K., Park, S. K., Kim, M. S., Lee, K., Lee, S. R., ...Park, E. (2019). Genetic diversity patterns and domestication origin of soybean. Theoretical and Applied Genetics, 132(4), 1179-1193. <a href='https://doi.org/10.1007/s00122-018-3271-7'>doi.org/10.1007/s00122-018-3271-7</a></td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Jeong_Moon_2019/'>View Here</a></td>
 </tr>

 <tr>
   <td>GmHapMap Data</td>
   <td>Torkamaneh, D., Laroche, J., Valliyodan, B., O'donoughue, L., Cober, E., Rajcan, I., ...Belzile, F. (2019). Soybean Haplotype Map (GmHapMap): A Universal Resource for Soybean Translational and Functional Genomics. BioRxiv, 534578. <a href='https://doi.org/10.1101/534578'>doi.org/10.1101/534578</a> </td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Torkamaneh_Laroche_2019/'>View Here</a></td>
 </tr>

 <tr>
   <td>374 US and Brazilian accessions </td>
   <td>Wei, W., Mesquita, A. C. O., Figueir√É, A. D. A., Wu, X., Manjunatha, S., Wickland, D. P., ...Clough, S. J. (2017). Genome-wide association mapping of resistance to a Brazilian isolate of Sclerotinia sclerotiorum in soybean genotypes mostly from Brazil. BMC genomics, 18(1), 849. <a href='https://doi.org/10.1186/s12864-017-4160-1'>doi.org/10.1186/s12864-017-4160-1</a> </td>
   <td><a href='https://data.soybase.org/Glycine/max/diversity/Wm82.gnm2.div.Wei_Mesquita_2017/'>View Here</a></td>
 </tr>
</table>
<p> * please cite accordingly </p> 
</div>
</div>
<br>

{% assign iframe_style="display: flex; width: 100%; height: 100vh; flex-direction: column;" %}
<iframe id="myiframe" src="{{ page.iframe_url }}" style="{{ iframe_style }}"></iframe>

<script language="javascript" src="../../assets/js/utils.js">
    var iframe = document.getElementById('myiframe');
        iframe.src = iframe.src + window.location.search;
</script>


